---
title: ATGCs (Alignable Tight Genomic Clusters)
description: A catalog of closely related prokaryotic genomes
published: true
date: 2020-04-29T14:51:58.774Z
tags: 
---

# ATGCs (Alignable Tight Genomic Clusters)

> ATGCs are a resource for micro- and macro-evolutionary studies of Bacteria and Archaea - in particular, pan-genomic studies of genes shared across multiple of these closely-related organisms. The major benefit of ATGCs is the large amount of pre-computed data, especially ATGC-COGs, multiple sequence alignments, and pairwise codeML evolutionary measurements. Examples include horizontal gene transfer (e.g., antibiotic resistance), gene neighborhood rearrangements (transposition, prophage integration), synonymous and nonsynonymous codon substitution models, phylogenetic inheritance, and many others
{.is-info}

## Website

- [ATGCs *Main page*](http://dmk-brain.ecn.uiowa.edu/ATGC/atgc_list.html)
{.links-list}

## Notes from users 
The ATGC list shows all the ATGCs available to view in the left column, along with additional data for each ATGC  to help identify ATGCs the user might want to study. The ATGC list can be downloaded as a CSV file and viewed in other programs.


## Ease of use 
The web page is not pleasant to look at but the data is pre-computed and publicly available. A good starting point for comparative genomics projects.

## Citation

- Kristensen DM, Wolf YI, Koonin EV. [ATGC database and ATGC-COGs: an updated resource for micro- and macro-evolutionary studies of prokaryotic genomes and protein family annotation](http://www.ncbi.nlm.nih.gov/pubmed/28053163). Nucleic Acids Res. 2017 Jan;45(Database issue):D210-8.
{.grid-list}