---
title: Computational Microbiology Laboratory Motif Locator
description: Uses aligned set of DNA sequence motifs (no gaps, all motifs of the same length) and finds similar motifs in the analyzed sequence. Uses position specific score matrix (PSSM) representation of the motif.
published: true
date: 2020-05-11T19:59:13.470Z
tags: tool, sequence analysis, 2008, motif
---

# Computational Microbiology Laboratory Motif Locator

> This program takes as input a set of aligned DNA sequence motifs (e.g., a set of transcription factor binding sites) and finds similar motifs in a DNA sequence (e.g., a prokaryotic chromosome). The primary output is a set of coordinates in the analyzed DNA sequence of motifs similar to those in the alignment. These coordinates can be subsequently passed to other programs (r-scan statistics, pattern vicinity analysis) in order to provide additional information about the distribution of the matching motifs in the analyzed sequence and with respect to genes.
{.is-info}

## Website

- [CMBL Motif Locator *Main Page*](https://www.cmbl.uga.edu//software/motloc.html)
- [Brief Explanation *Help section*](https://www.cmbl.uga.edu////software/Motloc-guide.htm)
{.links-list}

## Notes from Users
- send you the results by e-mail.
- lots of options for the read outs.

## Ease of use
Easy to input, but results are hard to read and interprete. 

## Citation

- Mr√°zek, Xie, Guo, and Srivastava (2008) [AIMIE: A Web-based Environment for Detection and Interpretation of Significant Sequence Motifs in Prokaryotic Genomes.](https://www.ncbi.nlm.nih.gov/pubmed/18304933) Bioinformatics, in press.
{.grid-list}