---
title: Databases
description: Here you will found the link to the wiki's pages of databases 
published: true
date: 2020-09-10T17:54:06.790Z
tags: category
editor: markdown
dateCreated: 2020-04-29T20:55:11.424Z
---

# Databases

## Databases Related To Enzymes And Pathways

- [cpnDB *Manually curated, taxonomically broad collection of chaperonin sequences*](https://vdclab-wiki.herokuapp.com/en/databases/enzyme-pathway/cpnDB)
- [FuzDB *Fuzzy protein complexes based on experimental evidence*](https://vdclab-wiki.herokuapp.com/en/databases/enzyme-pathway/FuzDB)
- [InnateDB *Genes, proteins, interactions and signaling pathways involved in the innate immune response*](https://vdclab-wiki.herokuapp.com/en/databases/enzyme-pathway/InnateDB)
- [KEGG PATHWAY *Manually drawn pathway maps representing our knowledge on molecular interaction*](https://vdclab-wiki.herokuapp.com/en/databases/enzyme-pathway/KEGG-PATHWAY)
- [:star: MetaCyc *Curated database of experimentally elucidated metabolic pathways*](https://vdclab-wiki.herokuapp.com/en/databases/enzyme-pathway/MetaCyc)
- [:star: Pathguide *Provides an overview of more than 190 biological pathway and network databases*](https://vdclab-wiki.herokuapp.com/en/databases/enzyme-pathway/Pathguide)
- [PROSITE *Database of protein families and domains*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/ExPASy-PROSITE)
- [SABIO-RK *Curated database that contains information about biochemical reactions*](https://vdclab-wiki.herokuapp.com/en/databases/enzyme-pathway/SABIO-RK)
- [:star: SimpleClinVar *Web server to explore and retrieve gene and disease variants aggregated in ClinVar database*](https://vdclab-wiki.herokuapp.com/en/databases/enzyme-pathway/SimpleClinVar)
{.links-list}

## Database Searches

- [Batch-Entrez *Entrez accession numbers or other identifiers, Batch Entrez downloads the corresponding records*](https://vdclab-wiki.herokuapp.com/en/databases/database_searches/Batch-Entrez)
- [Bio.tools *Comprehensive registry of software and databases*](https://vdclab-wiki.herokuapp.com/databases/database_searches/biotools/)
- [ProDom *Comprehensive set of protein domain families automatically generated from the UniProt KD*](https://vdclab-wiki.herokuapp.com/en/databases/database_searches/ProDom)
{.links-list}

## Data Integration Databases


- [:star: BioCyc *Collection of 16,822 Pathway/Genome Databases (PGDBs)*](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/BioCyc)
- [:star: BRENDA ENZYMES *Main collection of enzyme functional data available*](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/BRENDA-ENZYMES)
- [:star: CATH *Classification of protein structures downloaded from the Protein Data Bank*](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/CATH)
- [:star: DEG *Database of Essential Genes*](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/DEG)
- [DisProt *Database of intrinsically disordered proteins*](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/DisProt)
- [EuPathDB *Eukaryotic pathogens database*](https://vdclab-wiki.herokuapp.com/databases/data-integration/EuPathDB/)
- [ExPASy ENZYME *Repository of enzymes and information relative to the nomenclature of enzymes*](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/ExPASy-ENZYME)
- [:star: FungiDB *Fungus database*](https://vdclab-wiki.herokuapp.com/databases/data-integration/FungiDB/)
- [FungiPath *Orthologous proteins in fungal genomes database*](https://vdclab-wiki.herokuapp.com/databases/data-integration/FUNGIpath/)
- [GenomeNet Database *Database of integrated genomes*](https://vdclab-wiki.herokuapp.com/databases/data-integration/genomenet/)
- [:star: JGI GOLD *Genomes Online Database*](https://vdclab-wiki.herokuapp.com/databases/data-integration/JGI-GOLD/)
- [PDBe-KB *Community-driven resource managed by the PDBe team*](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/PDBe-KB)
- [:star: QuartataWeb *User-friendly server developed for polypharmacological and chemogenomics analyses*](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/QuartataWeb)
- [Reactome *Signaling and metabolic molecules and their relations organized into biological pathways*](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/Reactome)
- [SCOP2 *Description of the structural and evolutionary relationships between all proteins whose structure is known*](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/SCOP2)
- [:star: SCOPe *Database developed at the Berkeley Lab and UC Berkeley*](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/SCOPe)
- [:star: SEED Viewer *Allows users to access the latest curated datasets for comparative genomics*](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/SEED-Viewer)
{.links-list}

## General Databases

- [cBiT *Repository that offers biomaterial-based transcriptomics data together with biomaterial metadata*](https://vdclab-wiki.herokuapp.com/en/databases/general_databases/cBiT)
- [:star: Database_Commons *Provides users with access to a collection of publicly available biological databases*](https://vdclab-wiki.herokuapp.com/en/databases/general_databases/Database_Commons)
- [:star: DoriC *A database of oriC regions in bacterial genomes*](https://vdclab-wiki.herokuapp.com/en/databases/general_databases/DoriC)
- [:star: ECOD *Hierarchical classification of protein domains according to their evolutionary relationships*](https://vdclab-wiki.herokuapp.com/en/databases/general_databases/ECOD)
- [:star: InterPro *Functional analysis of proteins by classifying them into families and predicting domains*](https://vdclab-wiki.herokuapp.com/en/databases/general_databases/InterPro)
- [JGI *Joint Genome Institute - Main webiste*](https://vdclab-wiki.herokuapp.com/en/databases/general_databases/JGI)
- [JGI Genome Portal *JGI genomic databases and analytical tools*](https://vdclab-wiki.herokuapp.com/en/databases/general_databases/Genome-Portal) 
- [:star: KBase *Large-scale bioinformatics system that enables users to upload their own data*](https://vdclab-wiki.herokuapp.com/databases/general_databases/KBase/)
- [:star: NCBI *All Resources*](https://vdclab-wiki.herokuapp.com/en/resources/general_resources/NCBI-All-Resources)
- [:star: NPAtlas *Designed to cover all microbially-derived natural products published*](https://vdclab-wiki.herokuapp.com/en/databases/general_databases/NPAtlas)
- [:star: RPFdb *Public resource for hosting, analyzing and visualizing ribosome profiling (Ribo-seq) data*](https://vdclab-wiki.herokuapp.com/en/databases/general_databases/RPFdb)
- [:star: UniProt *Comprehensive resource for protein sequence and annotation data*](https://vdclab-wiki.herokuapp.com/en/databases/general_databases/UniProt)
{.links-list}

### Eukaryote Databases

- [SIGNOR *Organizes and stores in a structured format signaling information in mammals*](https://vdclab-wiki.herokuapp.com/en/databases/eukaryote_databases/SIGNOR)
{.links-list}

#### Plant Databases

- [Gramene *Integrated data resource for comparative functional genomics in crops and model plant species*](https://vdclab-wiki.herokuapp.com/en/databases/plant_databases/Gramene)
- [JGI-Phytozome *Plant Comparative Genomics portal of the Department of Energy's Joint Genome Institute*](https://vdclab-wiki.herokuapp.com/en/databases/plant_databases/JGI-Phytozome)
- [PlantGDB *Resources for comparative plant genomics*](https://vdclab-wiki.herokuapp.com/en/databases/plant_databases/PlantGDB)
- [PlantPromoterDB *Provides comprehensive transcriptional regulatory elements informations*](https://vdclab-wiki.herokuapp.com/en/databases/plant_databases/PlantPromoterDB)
- [PlnTFDB *Identification and cataloging all Plant genes involved in transcriptional control*](https://vdclab-wiki.herokuapp.com/en/databases/plant_databases/PlnTFDB)
- [Plprot *Plastid protein database-integrates data from large scale proteome analyses*](https://vdclab-wiki.herokuapp.com/en/databases/plant_databases/plprot)
- [The Plant Proteome DataBase *Database for Arabidopsis thaliana and maize (Zea mays)*](https://vdclab-wiki.herokuapp.com/en/databases/plant_databases/PPDB)
{.links-list}

#### Organism Specific Databases 

- [:star: Arabidopsis Information Resource (TAIR) *database of genetic and molecular biology data for A. thaliana*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/TAIR)
- [ALGAEpath *Coexpression Analysis on algae*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/ALGAEpath-Coexpression-Analysis)
- [BioXpress *Gene/miRNA expression in cancer database with expression levels mapped to genes or miRNAs*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/BioXpress)
- [BioMuta *Single-nucleotide variation (SNV) in cancer and disease association database*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/BioMuta)
- [:star: CellMap *Storing and visualizing genetic interactions in S. cerevisiae*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/CellMap)
- [ChlamyNet Clusters *Exploration of gene co-expression network of Chlamydomonas transcriptome*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/ChlamyNet-Clusters)
- [:star: HumanMine *An integrated database of Homo sapiens genomic data*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/HumanMine)
- [MaizeGDB *Focused on the crop plant and model organism Zea mays*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/MaizeGDB)
- [:star: neXtProt *Exploring the universe of human proteins*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/neXtProt)
- [OncoMX *Exploring Cancer Biomarkers in the Context of Related Cancer and Healthy Data*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/OncoMX)
- [:star: PomBase *Scientific resource for fission yeast*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/PomBase)
- [:star: Saccharomyces Genome Database *Comprehensive integrated biological information for the budding yeast S. cerevisiae*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/SGD)
- [:star: SEEK *Computational gene co-expression search engine*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/SEEK)
- [:star: SPELL *Query-driven search engine for large gene expression microarray compendia on S. cerevisiae*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/SPELL)
{.links-list} 

### Prokaryote Databases

- [ATGCs *Resource for micro- and macro-evolutionary studies of Bacteria and Archaea*](https://vdclab-wiki.herokuapp.com/en/databases/general_databases/ATCGs)
- [:star: JGI-IMG/Microbes *Community resource for analysis and annotation of genome*](https://vdclab-wiki.herokuapp.com/en/databases/bacterial_databases/JGI-IMG)
- [NCBI *Microbial Database*](https://vdclab-wiki.herokuapp.com/databases/general_databases/NCBI-genomes-microbes/)
{.links-list}

#### Bacterial Databases

- [ASAP *Relational database distributes genome sequence data and gene expression data*](https://vdclab-wiki.herokuapp.com/en/databases/bacterial_databases/ASAP)
- [:star: BacMap *An Interactive Atlas for Exploring Bacterial Genomes*](https://vdclab-wiki.herokuapp.com/en/databases/bacterial_databases/BacMap)
- [:star: eLMSG *Intended to integrate microbial systematics, genomics and phenomics*](https://vdclab-wiki.herokuapp.com/en/databases/bacterial_databases/eLMSG)
- [OperonDB *Predicting operons in microbial genomes*](https://vdclab-wiki.herokuapp.com/en/databases/bacterial_databases/OperonDB)
- [:star: PATRIC *Data and analysis tools to support biomedical research on bacterial infectious diseases*](https://vdclab-wiki.herokuapp.com/databases/bacterial_databases/patric/)
- [PRODORIC *Integrated approach to provide information about molecular networks in prokaryotes*](https://vdclab-wiki.herokuapp.com/en/databases/bacterial_databases/PRODORIC)
- [:star: PubSEED *SEED is a constantly updated integration of genomic data*](https://vdclab-wiki.herokuapp.com/en/databases/bacterial_databases/pubseed)
- [xBASE *Genome database aimed at helping laboratory-based bacteriologists*](https://vdclab-wiki.herokuapp.com/en/databases/bacterial_databases/xBASE)
{.links-list}

##### Organism Specific Databases

- [:star: AureoWiki *Repository of the Staphylococcus aureus research & annotation*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/AureoWiki)
- [CyanoEXpress *Curated genome-wide expression data for the cyanobacterium Synechocystis sp. PCC 6803*](https://vdclab-wiki.herokuapp.com/databases/organism_specific/CyanoEXpress/)
- [EchoBASE *Integrated post-genomic database for E.coli*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/EchoBASE)
- [:star: EcoCyc *Scientific database for the bacterium Escherichia coli K-12 MG1655*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/EcoCyc)
- [:star: EcoliNet *Probabilistic functional gene network for Escherichia coli*](https://vdclab-wiki.herokuapp.com/en/tools/gene-networks/EcoliNet)
- [PHIDIAS *Data related to pathogen-host interactions for pathogens*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/PHIDIAS)
- [:star: Pseudomonas Genome DB *Genome annotation and genome analysis data of Pseudomonas*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/Pseudomonas-Genome-DB)
- [SalCom *HintonLab's Salmonella enterica serovar Typhimurium Gene Expression Compendium*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/SalCom)
- [SToPSdb *Sub-cellular Topology of Polypeptides in Streptomyces*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/SToPSdb)
- [:star: SubtiWiki *Relational database of B. subtilis*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/SubtiWiki)
- [:star: SynWiki *Dedicated to the current generated functional annotation of the synthetic organism JCVI-syn3.0/syn3A*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/SynWiki)
- [:star: TB Genomes Database *An integrated platform for Tuberculosis research*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/TBDB)
{.links-list}

#### Archaeal Databases

- [:star: UCSC Archaeal Genome Browser *Basic gene annotation of 100 archaeal species*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/UCSC-AGB)
{.links-list}

### Virus Databases

- [:star: JGI-IMG/Virus *Community resource for analysis and annotation of genome*](https://vdclab-wiki.herokuapp.com/databases/bacterial_databases/JGI-IMG-VR/)
- [GiantVirus *Mimivirus and other large DNA viruses*](https://vdclab-wiki.herokuapp.com/en/databases/virus_databases/GiantVirus)
- [RVDB *Enhancing virus detection using next-generation sequencing (NGS) technologies*](https://vdclab-wiki.herokuapp.com/en/databases/virus_databases/RVDB)
- [PhagesDB *Characterization, and genomics of phages that infect bacterial hosts within the phylum Actinobacteria*](https://vdclab-wiki.herokuapp.com/en/databases/virus_databases/PhagesDB)
- [pVOGs *Orthologous groups shared among complete genomes of viruses that infect bacteria and archaea*](https://vdclab-wiki.herokuapp.com/en/databases/virus_databases/pVOGs)
- [ViPR *Information for human pathogenic viruses*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/ViPR)
- [VirusHostDB *Linking virus genomes with host taxonomy*](https://vdclab-wiki.herokuapp.com/en/databases/virus_databases/VirusHostDB)
- [VOGDB *Virus orthologous groups*](https://vdclab-wiki.herokuapp.com/en/databases/virus_databases/VOGDB)
{.links-list}

## Metabolic Pathways

- [ModelSEED *Resource for the reconstruction, exploration, comparison, and analysis of metabolic models*](https://vdclab-wiki.herokuapp.com/en/databases/metabolic_pathways/ModelSEED)
{.links-list}

## Microarray and RNAseq Databases

- [:star: ATTED-II *Coexpression database for plants to discover relationships of unknown genes within a species*](https://vdclab-wiki.herokuapp.com/en/databases/microarray-rnaseq/ATTED-II)
- [ArrayExpress *Functional Genomics Data stores data from high-throughput functional genomics experiments*](https://vdclab-wiki.herokuapp.com/en/databases/microarray-rnaseq/ArrayExpress)
- [Colombos *Exploring and analyzing comprehensive organism-specific cross-platform expression of Bacteria*](https://vdclab-wiki.herokuapp.com/en/databases/microarray-rnaseq/Colombos)
- [:star: COXPRESdb *Database providing coexpression information for 11 animal species*](https://vdclab-wiki.herokuapp.com/en/databases/microarray-rnaseq/COXPRESdb)
- [eFP *Schematic representation of data for Arabidopsis seedlings*](https://vdclab-wiki.herokuapp.com/en/databases/microarray-rnaseq/eFP)
- [GenExpDB *Repository for E. coli gene expression data*](https://vdclab-wiki.herokuapp.com/en/databases/microarray-rnaseq/GenExpDB)
- [Harmonizome *Comprehensive resource of knowledge about genes and proteins*](https://vdclab-wiki.herokuapp.com/en/databases/microarray-rnaseq/Harmonizome)
- [NCBI *Gene Expression Omnibus (GEO)*](https://vdclab-wiki.herokuapp.com/en/databases/microarray-rnaseq/NCBI-GEO)
- [qTeller for MaizeGDB *Comparative RNA-seq expression platform for Zea mays genome*](https://vdclab-wiki.herokuapp.com/en/databases/microarray-rnaseq/qTeller-for-MaizeGDB)
{.links-list}

## Ortholog Databases

- [KEGG Ortholog Cluster *Database of ortholog clusters (OCs) based on the whole genome comparison*](https://vdclab-wiki.herokuapp.com/en/databases/ortholog/KEGG-OC)
- [:star: KofamKOALA *Assigns K numbers to the user's sequence data by HMMER/HMMSEARCH against KOfam*](https://vdclab-wiki.herokuapp.com/en/databases/ortholog/KofamKOALA)
- [MBGD *Workbench system for comparative analysis of completely sequenced microbial genomes*](https://vdclab-wiki.herokuapp.com/en/databases/ortholog/MBGD)
- [OrthoDB *OrthoDB presents a catalog of orthologous protein-coding genes*](https://vdclab-wiki.herokuapp.com/en/databases/ortholog/OrthoDB)
- [:star: OrthoMCL *OrthoMCL is a genome-scale algorithm for grouping orthologous protein sequences*](https://vdclab-wiki.herokuapp.com/en/databases/ortholog/OrthoMCL)
- [:star: OrthoVenn2 *Platform for comparison and annotation of orthologous gene clusters among multiple species*](https://vdclab-wiki.herokuapp.com/en/visualization/venn-diagram/OrthoVenn2)
- [WebNetCoffee *Web server based on the algorithm of NetCoffee for aligning multiple PPI networks*](https://vdclab-wiki.herokuapp.com/en/databases/ortholog/WebNetCoffee)
{.links-list}

## Phenotypes And Fitness Databases

- [:star: Chemgen *Searching of growth data (strains/condition) of various E. coli*](https://vdclab-wiki.herokuapp.com/en/databases/phenotypes-fitness/chemgen)
- [:star: CTD *Advance understanding about how environmental exposures affect human health*](https://vdclab-wiki.herokuapp.com/en/databases/phenotypes-fitness/CTD)
- [:star: FitDB *Searchable database of quantitative chemical-genetic interactions*](https://vdclab-wiki.herokuapp.com/en/databases/phenotypes-fitness/FitDB)
- [:star: FitnessBrowser *Browse thousands of genome-wide fitness experiments from diverse bacteria*](https://vdclab-wiki.herokuapp.com/en/databases/phenotypes-fitness/FitnessBrowser)
- [GenAtlas *Scientific and clinical digests on genes and diseases*](https://vdclab-wiki.herokuapp.com/en/databases/phenotypes-fitness/GenAtlas)
- [:star: GeneCards *The Human Gene Database*](https://vdclab-wiki.herokuapp.com/en/databases/phenotypes-fitness/GeneCards)
- [MorphoBank *Database of peer-reviewed morphological matrices*](https://vdclab-wiki.herokuapp.com/en/databases/phenotypes-fitness/MorphoBank)
- [:star: OGEE *Aims to enhance our understanding of the essentiality of genes*](https://vdclab-wiki.herokuapp.com/en/databases/phenotypes-fitness/OGEE)
- [OMIM *Comprehensive, authoritative compendium of human genes and genetic phenotypes*](https://vdclab-wiki.herokuapp.com/en/databases/phenotypes-fitness/OMIM)
- [:star: VarSite *Aims to annotate known disease-associated variants in human genes with structural information*](https://vdclab-wiki.herokuapp.com/en/databases/phenotypes-fitness/VarSite)
{.links-list}

## Post-Translational Modification Databases

- [iPTMnet *Integrated understanding of protein post-translational modifications*](https://vdclab-wiki.herokuapp.com/en/databases/post-translational-modification/iPTMnet)
{.links-list}

## Protein-Protein Interaction Databases

- [:star: APID *Agile Protein Interactomes DataServer gives a thorough collection of protein interactomes*](https://vdclab-wiki.herokuapp.com/en/databases/protein-protein-interaction/APID)
- [CellPhoneDB *Repository of curated receptors, ligands and their interactions*](https://vdclab-wiki.herokuapp.com/en/databases/protein-protein-interaction/CellPhoneDB)
- [DIP *Catalogs experimentally determined interactions between proteins*](https://vdclab-wiki.herokuapp.com/en/databases/protein-protein-interaction/DIP)
- [ChiPPI *Chimeric Protein-protein Interaction Server*](https://vdclab-wiki.herokuapp.com/en/databases/protein-protein-interaction/ChiPPI)
- [Emili Lab Database *Emili Lab research findings accessed via public databases*](https://vdclab-wiki.herokuapp.com/en/databases/protein-protein-interaction/Emili-Lab-Database)
- [IntAct *Open source database system and analysis tools for molecular interaction data*](https://vdclab-wiki.herokuapp.com/en/databases/protein-protein-interaction/IntAct)
- [PhosphoSitePlus *Comprehensive information and tools for the study of protein post-translational modifications*](https://vdclab-wiki.herokuapp.com/en/databases/protein-protein-interaction/PhosphoSitePlus)
- [Stitch *Database of known and predicted interactions between chemicals and proteins*](https://vdclab-wiki.herokuapp.com/en/databases/protein-protein-interaction/Stitch)
- [:star: ComplexPortal *Curated encyclopaedic resource of macromolecular complexes from a number of key model organisms*](https://vdclab-wiki.herokuapp.com/en/databases/protein-protein-interaction/ComplexPortal)
{.links-list}

## Transcription Factor Databases

- [CollecTF *Database of transcription factor binding sites in the Bacteria domain*](https://vdclab-wiki.herokuapp.com/en/databases/transciription_factor/CollecTF)
- [:star: RegPrecise *database for capturing, visualization and analysis of transcription factor regulons*](https://vdclab-wiki.herokuapp.com/en/databases/transciription_factor/RegPrecise)
- [:star: RegulonDB *Database on transcriptional regulation in Escherichia coli K-12 containing knowledge*](https://vdclab-wiki.herokuapp.com/en/databases/transciription_factor/RegulonDB)
{.links-list}

## RNA Database

- [:star: 3dmodmap *Database of maps showing the sites of modified rRNA nucleotides in 3D models*](https://vdclab-wiki.herokuapp.com/en/databases/RNA-database/3dmodmap)
- [:star: GtRNAdb *tRNA database contains tRNA gene predictions made by the program tRNAscan-SE*](https://vdclab-wiki.herokuapp.com/en/databases/tRNA-database/GtRNAdb)
- [NONCODE *Integrated knowledge database dedicated to non-coding RNAs (excluding tRNAs and rRNAs)*](https://vdclab-wiki.herokuapp.com/en/databases/RNA-database/NONCODE)
- [scRNASeqDB *Curated public single-cell gene expressing profiles for the wider research community*](https://vdclab-wiki.herokuapp.com/en/databases/RNA-database/scRNASeqDB)
- [:star: The RNA Modification Database *Listing of posttranscriptionally modified nucleosides from RNA*](https://vdclab-wiki.herokuapp.com/en/databases/RNA-database/The-RNA-Modification-Database)
- [:star: T-Psi-C *Allows you to search across all available tRNA sequences*](https://vdclab-wiki.herokuapp.com/en/databases/RNA-database/T-Psi-C)
- [:star: UCSC GtRNAdb *tRNAscan-SE analysis of complete genomes*](https://vdclab-wiki.herokuapp.com/en/databases/RNA-database/UCSC-GtRNAdb)
{.links-list}

## Unknown Gene/Enzyme Databases

- [ADOMETA *Bioinformatics resource designed to predict genes for orphan metabolic activities*](https://vdclab-wiki.herokuapp.com/en/databases/unknown-enzyme/ADOMETA)
- [:star: ORENZA *Accurate and up to date list of Enzyme Activities for which no sequences are available*](https://vdclab-wiki.herokuapp.com/en/databases/unknown-enzyme/ORENZA)
- [:star: Orphan Enzyme Project *Associate an amino-acid sequence with every known orphan enzyme*](https://vdclab-wiki.herokuapp.com/en/databases/unknown-enzyme/Orphan-Enzymes)
{.links-list}