---
title: The Database for Annotation, Visualization and Integrated Discovery (DAVID)
description:  DAVID provides a comprehensive set of functional annotation tools for investigators to understand biological meaning behind large list of genes.
published: true
date: 2020-05-30T20:31:14.387Z
tags: annotation tools, database, data visualization
---

# The Database for Annotation, Visualization and Integrated Discovery (DAVID)

The Database for Annotation, Visualization and Integrated Discovery (DAVID ) v6.8 comprises a full Knowledgebase update to the sixth version of our original web-accessible programs. DAVID now provides a comprehensive set of functional annotation tools for investigators to understand biological meaning behind large list of genes.

For any given gene list, DAVID tools are able to:

&NewLine;

     - Identify enriched biological themes, particularly GO terms
     - Discover enriched functional-related gene groups
     - Cluster redundant annotation terms
     - Visualize genes on BioCarta & KEGG pathway maps
     - Display related many-genes-to-many-terms on 2-D view.
     - Search for other functionally related genes not in the list
     - List interacting proteins
     - Explore gene names in batch
     - Link gene-disease associations
     - Highlight protein functional domains and motifs
     - Redirect to related literatures
     - Convert gene identifiers from one type to another.
     - And more
     
&NewLine;

{.is-info}



## Website

- [DAVID *Main Page*](https://david.ncifcrf.gov/)
{.links-list}

## User Notes
- Mapped identifiers requires additional review/checking, removal of duplicates or incorrectly mapped IDs (specific attention should be applied to the listed organism identities, especially for duplicate gene/protein entries)
- Excellent breadth of mapping (e.g., if UniProt's ID mapping tool falls short in coverage, failing to map a large number of IDs), hence the above-mentioned need for additional review
- Enrichment (as with any form of enrichment analysis) requires an understanding of how to interpret this character of heuristic output; respect that lack of empiricism, using caution in application of the results

## Citations

- Huang DW, Sherman BT, Lempicki RA. [Systematic and integrative analysis of large gene lists using DAVID Bioinformatics Resources.](https://www.nature.com/articles/nprot.2008.211) Nature Protoc. 2009;4(1):44-57.
- Huang DW, Sherman BT, Lempicki RA. [Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists.](https://academic.oup.com/nar/article/37/1/1/1026684) Nucleic Acids Res. 2009;37(1):1-13.
{.grid-list}