---
title: VDC Favorites
description: Selection of the favorite website of the vdc lab
published: true
date: 2020-08-07T01:10:16.972Z
tags: reference, resource reference
editor: markdown
---

# VDC Favorite

> Here you can find a selection of the tools and website presented on this wiki. The selection is based on the favorite websites of Valérie de Crécy-Lagard.
{.is-info}

# Alignment

## Editing Alignments 

- [GUIDANCE2 *Weighting, filtering or masking unreliably aligned positions in sequence alignments before subsequent analysis*](https://vdclab-wiki.herokuapp.com/en/alignment/editing/GUIDANCE2)
{.links-list}

## Multiple Sequence Alignment

- [EMBL-EBI *Multiple Sequence Alignment*](https://vdclab-wiki.herokuapp.com/en/alignment/multiple-alignment/EMBL-EBI-MSA)
- [MAFFT *Multiple alignment program for amino acid or nucleotide sequences*](https://vdclab-wiki.herokuapp.com/en/alignment/multiple-alignment/MAFFT)
- [MultAlin *MSA from a group of related sequences using progressive pairwise alignments*](https://vdclab-wiki.herokuapp.com/en/alignment/multiple-alignment/MultAlin)
- [T-COFFEE *Align Protein, DNA and RNA sequences*](https://vdclab-wiki.herokuapp.com/en/alignment/multiple-alignment/T-COFFEE)
{.links-list}

## Two Sequences Alignment

- [Dotlet JS *Comparing sequences by the diagonal plot method*](https://vdclab-wiki.herokuapp.com/en/alignment/two-sequences-alignment/Dotlet)
- [Pairwise Alignment *Basic protein pairwise alignment tool*](https://vdclab-wiki.herokuapp.com/en/alignment/two-sequences-alignment/Pairwise-Alignment)
- [PRSS *Evaluates the significance of a protein sequence alignment*](https://vdclab-wiki.herokuapp.com/en/alignment/two-sequences-alignment/PRSS)
{.links-list}

# Annotation

## General Annotation Pipeline

- [GeneMark *Web page hosting families of gene prediction programs*](https://vdclab-wiki.herokuapp.com/en/annotation/general_annotation/GeneMark)
- [IslandViewer *Integrates four different genomic island prediction methods: IslandPick, IslandPath-DIMOB, SIGI-HMM, and Islander*](https://vdclab-wiki.herokuapp.com/en/annotation/general_annotation/IslandViewer)
- [ORFfinder *Searches for open reading frames (ORFs) in the DNA sequence you enter*](https://vdclab-wiki.herokuapp.com/en/annotation/general_annotation/ORFfinder)
{.links-list}

### Prokaryotic Genome Annotation Pipelines

- [KEGG-Annotation *Database contains three main components for genome/metagenome annotation*](https://vdclab-wiki.herokuapp.com/en/annotation/general_annotation/KEGG-Annotation)
- [proGenomes2 *Consistently annotated bacterial and archaeal genomes*](https://vdclab-wiki.herokuapp.com/annotation/prokaryotic/proGenomes2/)
{.links-list}

# Databases

## Databases Related To Enzymes And Pathways

- [MetaCyc *Curated database of experimentally elucidated metabolic pathways*](https://vdclab-wiki.herokuapp.com/en/databases/enzyme-pathway/MetaCyc)
- [Pathguide *Provides an overview of more than 190 biological pathway and network databases*](https://vdclab-wiki.herokuapp.com/en/databases/enzyme-pathway/Pathguide)
{.links-list}

## Data Integration Databases

- [BioCyc *Collection of 16,822 Pathway/Genome Databases (PGDBs)*](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/BioCyc)
- [BRENDA ENZYMES *Main collection of enzyme functional data available*](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/BRENDA-ENZYMES)
- [CATH *Classification of protein structures downloaded from the Protein Data Bank*](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/CATH)
- [DEG *Database of Essential Genes*](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/DEG)
- [FungiDB *Fungus database*](https://vdclab-wiki.herokuapp.com/databases/data-integration/FungiDB/)
- [JGI GOLD *Genomes Online Database*](https://vdclab-wiki.herokuapp.com/databases/data-integration/JGI-GOLD/)
- [SCOPe *Database developed at the Berkeley Lab and UC Berkeley*](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/SCOPe)
- [SEED Viewer *Allows users to access the latest curated datasets for comparative genomics*](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/SEED-Viewer)
{.links-list}

## General databases

- [InterPro *Functional analysis of proteins by classifying them into families and predicting domains*](https://vdclab-wiki.herokuapp.com/en/databases/general_databases/InterPro)
- [KBase *Large-scale bioinformatics system that enables users to upload their own data*](https://vdclab-wiki.herokuapp.com/databases/general_databases/KBase/)
- [RPFdb *Public resource for hosting, analyzing and visualizing ribosome profiling (Ribo-seq) data*](https://vdclab-wiki.herokuapp.com/en/databases/general_databases/RPFdb)
- [UniProt *Comprehensive resource for protein sequence and annotation data*](https://vdclab-wiki.herokuapp.com/en/databases/general_databases/UniProt)
{.links-list}

### Eukaryote databases

#### Organism specific databases 

- [Arabidopsis Information Resource (TAIR) *database of genetic and molecular biology data for A. thaliana*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/TAIR)
- [CellMap *Storing and visualizing genetic interactions in S. cerevisiae*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/CellMap)
- [HumanMine *An integrated database of Homo sapiens genomic data*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/HumanMine)
- [neXtProt *Exploring the universe of human proteins*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/neXtProt)
- [PomBase *Scientific resource for fission yeast*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/PomBase)
- [Saccharomyces Genome Database *Comprehensive integrated biological information for the budding yeast S. cerevisiae*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/SGD)
- [SEEK *Computational gene co-expression search engine*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/SEEK)
- [SPELL *Query-driven search engine for large gene expression microarray compendia on S. cerevisiae*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/SPELL)
{.links-list}

### Prokaryote databases

- [JGI-IMG/Microbes *Community resource for analysis and annotation of genome*](https://vdclab-wiki.herokuapp.com/en/databases/bacterial_databases/JGI-IMG)
{.links-list}

#### Bacterial databases

- [PATRIC *Data and analysis tools to support biomedical research on bacterial infectious diseases*](https://vdclab-wiki.herokuapp.com/databases/bacterial_databases/patric/)
- [PubSEED *SEED is a constantly updated integration of genomic data*](https://vdclab-wiki.herokuapp.com/en/databases/bacterial_databases/pubseed)
{.links-list}

##### Organism specific databases

- [AureoWiki *Repository of the Staphylococcus aureus research & annotation*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/AureoWiki)
- [EcoCyc *Scientific database for the bacterium Escherichia coli K-12 MG1655*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/EcoCyc)
- [Pseudomonas Genome DB *Genome annotation and genome analysis data of Pseudomonas*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/Pseudomonas-Genome-DB)
- [SubtiWiki *Relational database of B. subtilis*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/SubtiWiki)
- [TB Genomes Database *An integrated platform for Tuberculosis research*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/TBDB)
{.links-list}

#### Archaeal databases

- [UCSC Archaeal Genome Browser *Basic gene annotation of 100 archaeal species*](https://vdclab-wiki.herokuapp.com/en/databases/organism_specific/UCSC-AGB)
{.links-list}

### Virus databases

- [JGI-IMG/Virus *Community resource for analysis and annotation of genome*](https://vdclab-wiki.herokuapp.com/databases/bacterial_databases/JGI-IMG-VR/)
{.links-list}

## Microarray and Rnaseq databases

- [ATTED-II *Coexpression database for plants to discover relationships of unknown genes within a species*](https://vdclab-wiki.herokuapp.com/en/databases/microarray-rnaseq/ATTED-II)
- [COXPRESdb *Database providing coexpression information for 11 animal species*](https://vdclab-wiki.herokuapp.com/en/databases/microarray-rnaseq/COXPRESdb)
{.links-list}

## Ortholog Databases

- [OrthoMCL *OrthoMCL is a genome-scale algorithm for grouping orthologous protein sequences*](https://vdclab-wiki.herokuapp.com/en/databases/ortholog/OrthoMCL)
{.links-list}

## Phenotypes And Fitness Databases

- [Chemgen *Searching of growth data (strains/condition) of various E. coli*](https://vdclab-wiki.herokuapp.com/en/databases/phenotypes-fitness/chemgen)
- [CTD *Advance understanding about how environmental exposures affect human health*](https://vdclab-wiki.herokuapp.com/en/databases/phenotypes-fitness/CTD)
- [FitDB *Searchable database of quantitative chemical-genetic interactions*](https://vdclab-wiki.herokuapp.com/en/databases/phenotypes-fitness/FitDB)
- [FitnessBrowser *Browse thousands of genome-wide fitness experiments from diverse bacteria*](https://vdclab-wiki.herokuapp.com/en/databases/phenotypes-fitness/FitnessBrowser)
- [GeneCards *The Human Gene Database*](https://vdclab-wiki.herokuapp.com/en/databases/phenotypes-fitness/GeneCards)
- [OGEE *Aims to enhance our understanding of the essentiality of genes*](https://vdclab-wiki.herokuapp.com/en/databases/phenotypes-fitness/OGEE)
{.links-list}

## Protein-Protein Interaction Databases

- [APID *Agile Protein Interactomes DataServer gives a thorough collection of protein interactomes*](https://vdclab-wiki.herokuapp.com/en/databases/protein-protein-interaction/APID)
{.links-list}

## Transcription Factor Databases

- [RegPrecise *database for capturing, visualization and analysis of transcription factor regulons*](https://vdclab-wiki.herokuapp.com/en/databases/transciription_factor/RegPrecise)
- [RegulonDB *Database on transcriptional regulation in Escherichia coli K-12 containing knowledge*](https://vdclab-wiki.herokuapp.com/en/databases/transciription_factor/RegulonDB)
{.links-list}

## RNA Database

- [GtRNAdb *tRNA database contains tRNA gene predictions made by the program tRNAscan-SE*](https://vdclab-wiki.herokuapp.com/en/databases/RNA-database/GtRNAdb)
- [T-Psi-C *Allows you to search across all available tRNA sequences*](https://vdclab-wiki.herokuapp.com/en/databases/RNA-database/T-Psi-C)
{.links-list}

## Unknown Gene/Enzyme Databases

- [ORENZA *Accurate and up to date list of Enzyme Activities for which no sequences are available*](https://vdclab-wiki.herokuapp.com/en/databases/unknown-enzyme/ORENZA)
- [Orphan Enzyme Project *Associate an amino-acid sequence with every known orphan enzyme*](https://vdclab-wiki.herokuapp.com/en/databases/unknown-enzyme/Orphan-Enzymes)
{.links-list}

# Literature Search

- [PaperBLAST *database of protein sequences that are linked to scientific articles*](https://vdclab-wiki.herokuapp.com/en/literature-search/PaperBLAST)
- [PubMed Advanced Search Builder *Builder to do advanced search on PubMed*](https://vdclab-wiki.herokuapp.com/en/literature-search/PubMed-Advanced-Search-Builder)
{.links-list}

# Omics Data

## Omics Database

- [GNPS *Mass spectrometry knowledge base for community-wide organization*](https://vdclab-wiki.herokuapp.com/en/omics-data/omics-data/GNPS)
{.links-list}

# Phylogeny

## Phylogeny Tools

- [IQ-TREE *Efficient software for phylogenomic inference*](https://vdclab-wiki.herokuapp.com/en/phylogeny/tools/IQ-TREE)
- [NGPhylogeny *Reconstructing and analysing phylogenetic relationships*](https://vdclab-wiki.herokuapp.com/en/phylogeny/tools/NGPhylogeny)
- [Simple Phylogeny *Provides access to phylogenetic tree generation methods from the ClustalW2*](https://vdclab-wiki.herokuapp.com/en/phylogeny/tools/Simple-Phylogeny)
{.links-list}

## Phylogenetic Distribution Tools

- [AnnoTree *Visualization of genome annotations across large phylogenetic trees*](https://vdclab-wiki.herokuapp.com/en/phylogeny/phylogenetic-distribution/AnnoTree)
- [OrthoInspector *Inference of orthologous relationships between protein coding-genes*](https://vdclab-wiki.herokuapp.com/phylogeny/phylogenetic-distribution/OrthoInspector/)
{.links-list}

# Platform

## Genome Browsers Within Integrated Platforms

- [Ensembl Bacteria *Browser for bacterial and archaeal genomes*](https://vdclab-wiki.herokuapp.com/en/platform/genome-browsers/EnsemblBacteria)
{.links-list}

## Integrated Prokaryotic Genome Analysis Platforms

- [MicrobesOnline *An integrated portal for comparative and functional genomics*](https://vdclab-wiki.herokuapp.com/en/platform/integrated-prokaryotic-genome-analysis/MicrobesOnline)
{.links-list}

# Prediction

## Predicting Protein/Protein Interaction

- [ArDock *Detect interaction sites by docking arbitrary proteins*](https://vdclab-wiki.herokuapp.com/ArDock)
- [Skolnick Tools Suite *Suite of tools designed, produced, and made available by the Skolnick Lab*](https://vdclab-wiki.herokuapp.com/en/prediction/protein-protein-interaction/Skolnick-Lab-Tools)
- [Zhou Group Tools *Suite of tools for prediction of protein-protein interactions*](https://vdclab-wiki.herokuapp.com/en/prediction/protein-protein-interaction/Zhou-Group-Tools)
{.links-list}

# Ressources

## General Resources

- [BioNumbers *Enable you to find in one minute any useful molecular biology number*](https://vdclab-wiki.herokuapp.com/en/resources/general_resources/BioNumbers)
- [EMBL-EBI *Services and tools, perform basic research and provide professional training in bioinformatics*](https://vdclab-wiki.herokuapp.com/en/resources/general_resources/EMBL-EBI)
- [EZBioCloud *ChunLab's public data and analytics portal*](https://vdclab-wiki.herokuapp.com/resources/general_resources/EZBioCloud/)
- [FAIRsharing *A curated, informative and educational resource on data and metadata standards*](https://vdclab-wiki.herokuapp.com/en/resources/general_resources/FAIRsharing)
- [NCBI *All Resources*](https://vdclab-wiki.herokuapp.com/en/resources/general_resources/NCBI-All-Resources)
- [NCBI *Search engine*](https://vdclab-wiki.herokuapp.com/en/resources/general_resources/NCBI-Search)
- [NAR Database Summary *Category list*](https://vdclab-wiki.herokuapp.com/resources/general_ressources/NAR-cat-list/)
- [RTH *Analysis of the mammalian genome for non-coding RNAs in relation to (inflammatory) diseases*](https://vdclab-wiki.herokuapp.com/en/resources/general_resources/RTH)
{.links-list}

## Eukaryotes Resources

- [BahlerLab *Tools for analyzing S. pombe genetics and genetic regulation*](https://vdclab-wiki.herokuapp.com/en/resources/eukaryotes/BahlerLab)
- [BLAT Search Genome *Quickly find sequences of 95% and greater similarity of length 25 bases or more*](https://vdclab-wiki.herokuapp.com/en/resources/eukaryotes/BLAT-Search-Genome)
{.links-list}

### Plant Resources

- [Phylogenes *Pre-computed phylogenetic trees of gene families alongside experimental gene function data*](https://vdclab-wiki.herokuapp.com/en/resources/plant/phylogenes)
{.links-list}

## Experimental Planning Resources

- [ApE *Tool that allows information and editing of plasmids, as well as tools for data visualization*](https://vdclab-wiki.herokuapp.com/en/resources/experimental-planning/ApE)
- [NEBcutter *Take DNA sequence and find the large, open reading frames using the E.coli genetic code*](https://vdclab-wiki.herokuapp.com/en/resources/experimental-planning/NEBcutter)
- [Primer3Plus *Pick primers from a DNA sequence*](https://vdclab-wiki.herokuapp.com/en/resources/experimental-planning/Primer3Plus)
- [Primer-BLAST *A tool for finding specific primers*](https://vdclab-wiki.herokuapp.com/en/resources/experimental-planning/Primer-BLAST)
{.links-list}

## Homology

- [Genoscope Microscope Phyloprofile *Allows to search for common OR specific genes between genomes*](https://vdclab-wiki.herokuapp.com/en/resources/homology/Genoscope-Microscope-Phyloprofile)
- [JGI-IMG-Phylogeny *Assess phylogenetic composition of a metagenome sample based*](https://vdclab-wiki.herokuapp.com/en/resources/homology/JGI-IMG-Phylogeny)
{.links-list}

## Protein Analysis Resources

- [ExPASy Proteomics *Bioinformatic resources for proteomics*](https://vdclab-wiki.herokuapp.com/en/resources/analysis/ExPASy-Proteomics)
{.links-list}

# Sequence Manipulation

## Changing format of sequences

- [Nucleic Acid Sequence Massager *Tool to manipulate sequence(s) in fasta format*](https://vdclab-wiki.herokuapp.com/en/sequence-manipulation/changing-format/NA-Sequence-Massager)
- [Sequence Manipulation Suite *Sequence Manipulation Suite of bioinformatics.org*](https://vdclab-wiki.herokuapp.com/en/sequence-manipulation/changing-format/sequence-manipulation-suite)
{.links-list}

# Structure

## Evolutionary Conserved Residues On Structure

- [ConSurf *Server for the Identification of Functional Regions in Proteins*](https://vdclab-wiki.herokuapp.com/en/structure/evolutionary_conserved_residues/ConSurf)
{.links-list}

## Ligand prediction

- [BindUP *Predicting DNA and RNA binding proteins using the electrostatic patches on the protein surface*](https://vdclab-wiki.herokuapp.com/en/structure/ligand-prediction/BindUP)
{.links-list}

## Modeling Protein Structures

- [GalaxyWEB *Web server for protein structure prediction, refinement, and related methods*](https://vdclab-wiki.herokuapp.com/en/structure/modeling-protein-structures/GalaxyWEB)
- [RaptorX *Protein structure prediction server*](https://vdclab-wiki.herokuapp.com/en/structure/modeling-protein-structures/RaptorX)
{.links-list}


## Structural Base Alignments

- [RNAcentral *The non-coding RNA sequence database*](https://vdclab-wiki.herokuapp.com/structure/structural_base_alignments/RNAcentral)
{.links-list}

## Structure Visualization Platforms 

- [Aquaria *Allows for the visualization of protein structure of various proteins from numerous species*](https://vdclab-wiki.herokuapp.com/en/structure/visualization-platforms/Aquaria)
- [iCn3D *Viewer for interactive viewing of three-dimensional macromolecular structures and chemicals*](https://vdclab-wiki.herokuapp.com/en/structure/visualization-platforms/iCn3D)
- [Phyre *Web-based services for protein structure prediction*](https://vdclab-wiki.herokuapp.com/en/structure/visualization-platforms/Phyre)
- [RCSB-PDB *Makes PDB data available at no charge to all data consumers*](https://vdclab-wiki.herokuapp.com/en/structure/visualization-platforms/RCSB-PDB)
- [SWISS-MODEL *Web-based integrated service dedicated to protein structure homology modelling*](https://vdclab-wiki.herokuapp.com/en/structure/visualization-platforms/SWISS-MODEL)
{.links-list}

# Tools

## Analysis Tools

- [CD-Search *Searching the Conserved Domain Database with protein or nucleotide query sequences*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/CD-Search)
- [Compute pI *Allows the computation of the theoretical pI (isoelectric point) and Mw (molecular weight)*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/Compute-pI)
- [ENDscript *Extracts and renders a comprehensive analysis of primary to quaternary protein structure information*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/ENDscript)
- [Enzyme Function Initiative (EFI) *Creating and interacting with sequence similarity networks*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/Enzyme-Function-Initiative)
- [ESPript *Program which renders sequence similarities and secondary structure information from aligned sequences*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/ESPript)
- [ETviewer *Relative rank of functional and structural importance among protein homologs sequence positions*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/ETviewer)
- [Genoscope - MicroScope *MicroScope offers many useful data-exploring functionalities*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/Genoscope-Microscope)
- [MolBiol-Tools *Online tools and resources for molecular biologists*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/MolBiol-Tools)
- [MEME-Suite *Discover novel motifs in collections of unaligned nucleotide or protein sequences*](https://vdclab-wiki.herokuapp.com/en/tools/sequence-logos/The-MEME-Suite)
- [MitoMiner *Database of mammalian mitochondrial localisation evidence, phenotypes, and diseases*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/MitoMiner)
- [NCBI *Conserved-Domains*](https://vdclab-wiki.herokuapp.com/tools/analysis/NCBI-Conserved-Domains/)
- [NPS@ *An interactive web server dedicated to protein sequence analysis.*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/NPS@)
- [Phobius *Prediction of transmembrane topology and signal peptides*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/Phobius)
- [PROSITE *Database of protein families and domains*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/ExPASy-PROSITE)
- [PSORTb *Precise bacterial localization prediction tool*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/PSORTb)
- [SignalP *Signal peptide and cleavage sites in gram+, gram- and eukaryotic amino acid sequences*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/SignalP)
- [SoftBerry *Tools for genomic using computational methods of high throughput biomedical data analysis*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/SoftBerry)
- [SynMap *Generates a syntenic dotplot between two organisms and identifies syntenic regions*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/SynMap)
- [TargetP *Subcellular location of proteins: mitochondrial, chloroplastic, secretory pathway*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/TargetP)
- [TMHMM *Prediction of transmembrane helices in proteins*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/TMHMM)
- [Zhang Lab *Web page dedicated to the research, publications, information of the Zhang Lab*](https://vdclab-wiki.herokuapp.com/tools/analysis/Zhang-Lab)
{.links-list}


## Alignment tools

- [SIAS *Calculates pairwise sequence identity and similarity from multiple sequence alignments*](https://vdclab-wiki.herokuapp.com/en/tools/alignment/SIAS)
{.links-list}

## Cloning Tools Platforms

- [Benchling *Electronic notebook and a suite of sequence design tools for free to academics*](https://vdclab-wiki.herokuapp.com/en/tools/cloning-platforms/Benchling)
{.links-list}


## Creating Or Finding Sequence Logos

- [Berkeley WebLogo *Designed to make the generation of sequence logos as easy and painless as possible*](https://vdclab-wiki.herokuapp.com/en/visualization/sequence-data-visualization/Berkeley-WebLogo)
- [Skylign *Creating logos representing both sequence alignments and profile hidden Markov models*](https://vdclab-wiki.herokuapp.com/en/tools/sequence-logos/Skylign)
- [WebLogo3 *Take multiple sequence alignments and generate a graphical representation of the consensus*](https://vdclab-wiki.herokuapp.com/en/tools/sequence-logos/WebLogo3)
{.links-list}

## Database Linker

- [bioDBnet-db2db *Database to Database Conversions*](https://vdclab-wiki.herokuapp.com/en/tools/database-linker/bioDBnet-db2db)
- [UniProt-Mapping *Very usuful tool when you want to swap IDs from a database to another*](https://vdclab-wiki.herokuapp.com/en/tools/database-linker/UniProt-Mapping)
{.links-list}

## Data formatting

- [Sequence Massager *Easy modification and editing of user-inputted nucleic acid sequences*](https://vdclab-wiki.herokuapp.com/en/tools/data-formatting/Sequence-Massager)
{.links-list}

## Gene finders

- [BLAST *Finds regions of local similarity between sequences*](https://vdclab-wiki.herokuapp.com/en/tools/gene-finders/BLAST)
- [HMMER *Fast and sensitive homology searches*](https://vdclab-wiki.herokuapp.com/tools/gene-finders/EMBL-EBI-HMMER/)
- [InterMine *Integrates biological data sources, making it easy to query and analyse data*](https://vdclab-wiki.herokuapp.com/en/tools/gene-finders/InterMine)
{.links-list}

## Gene Networks

- [EcoliNet *Probabilistic functional gene network for Escherichia coli*](https://vdclab-wiki.herokuapp.com/en/tools/gene-networks/EcoliNet)
- [GeneMANIA *Predict the functions of your favorite genes and gene sets*](https://vdclab-wiki.herokuapp.com/en/tools/gene-networks/GeneMANIA)
- [IMP *Analyze your experimental results in the functional context of gene-gene networks*](https://vdclab-wiki.herokuapp.com/en/tools/gene-networks/IMP)
{.links-list}

## Gene Ontology Enrichment Tools

- [GO Enrichment Analysis *Representation of our current scientific knowledge about the functions of genes*](https://vdclab-wiki.herokuapp.com/en/tools/gene_ontology/GO-Enrichment-Analysis)
{.links-list}

## Prediction

- [AACompIdent *Allows the identification of a protein from its amino acid composition*](https://vdclab-wiki.herokuapp.com/en/tools/prediction/AACompIdent)
- [COGcollator *Web server for analysis of distant relationships between homologous protein families*](https://vdclab-wiki.herokuapp.com/en/tools/prediction/COGcollator)
- [RNAmmer *Predicts 5s/8s, 16s/18s, and 23s/28s ribosomal RNA in full genome sequences*](https://vdclab-wiki.herokuapp.com/en/tools/prediction/RNAmmer)
- [tRNAscan-SE *Searching for tRNA genes in genomic sequence*](https://vdclab-wiki.herokuapp.com/en/tools/prediction/tRNAscan-SE)
{.links-list}

## Visualizing Pathways - Mapping Data To Pathways

- [PaintOmics *Integrative visualization of multiple omic datasets onto KEGG pathways*](https://vdclab-wiki.herokuapp.com/en/tools/pathways/PaintOmics)
- [PathView *Provides easy interactive access, and generates high quality, hyperlinked graphs*](https://vdclab-wiki.herokuapp.com/en/tools/pathways/PathView)
{.links-list}

# Trainings & Tutorials

## Computing & Programming

- [Bioinformatics Club Bioinformatics Course *Regular expression for Biology*](/trainings-tutorials/computing-programming/Bioinformatics-Club-Bioinformatics-Course/)
{.links-list}

## Tutorials

- [NCBI *Tutorial page*](https://vdclab-wiki.herokuapp.com/trainings-tutorials/tutorials/NCBI-tutorials/)
{.links-list}

# Visualization

## Data Visualization

- [Clustergrammar *Analyzing high-dimensional data as interactive hierarchically clustered heatmaps*](https://vdclab-wiki.herokuapp.com/en/visualization/data-visualization/Clustergrammar)
- [Heatmapper *Online application for visualizing various types of data as heat maps*](https://vdclab-wiki.herokuapp.com/en/visualization/data-visualization/Heatmapper)
- [Jalview *Program for multiple sequence alignment editing*](https://vdclab-wiki.herokuapp.com/en/visualization/data-visualization/Jalview)
{.links-list}

## Gene Neighborhood Figures Tools

- [GCTNG *Visualize the genomic context of a gene or a group of genes and their orthologous relationships*](https://vdclab-wiki.herokuapp.com/en/visualization/gene-neighborhood/GCTNG)
- [Gene Graphics *Application that allows for visually representations of physical gene neighborhoods*](https://vdclab-wiki.herokuapp.com/en/visualization/gene-neighborhood/Gene-Graphics)
{.links-list}

## Genome Browsers Within Integrated Platforms

- [GenomeProjector *Zoomable user interface for molecular biology*](https://vdclab-wiki.herokuapp.com/en/visualization/genome-browsers/GenomeProjector)
{.links-list}

## Miscellaneous

- [IBS *Assisting experimentalists in drawing publication-quality diagrams of both protein and nucleotide sequences*](https://vdclab-wiki.herokuapp.com/en/visualization/miscellaneous/IBS)
{.links-list}

## Motifs

- [Seq2Logo *Sequence logo generator*](https://vdclab-wiki.herokuapp.com/en/visualization/motifs/Seq2Logo)
{.links-list}

## Sequence Data Visualization

- [MView *Reformats the results of a sequence database search or a multiple alignment*](https://vdclab-wiki.herokuapp.com/en/visualization/sequence-data-visualization/MView)
- [Two Sample Logos *Calculates and visualizes differences between two sets of aligned samples of amino acids or nucleotides*](https://vdclab-wiki.herokuapp.com/en/visualization/sequence-data-visualization/TSL)
{.links-list}

## Structure Visualization Platforms 

- [Cn3D *Allows you to view 3-dimensional structures from NCBI's Entrez Structure database*](https://vdclab-wiki.herokuapp.com/en/visualization/structure_platform/Cn3D)
- [JPred *A Protein Secondary Structure Prediction Server*](https://vdclab-wiki.herokuapp.com/en/visualization/visualization-platforms/JPred)
{.links-list}

## Tools to Make Figures

- [EvolView *Easy-to-use online tool for displaying, managing and customizing phylogenetic trees*](https://vdclab-wiki.herokuapp.com/en/visualization/tools-to-make-figures/EvolView)
- [iTOL *Online tool for the display, annotation and management of phylogenetic trees*](https://vdclab-wiki.herokuapp.com/en/visualization/tools-to-make-figures/iTOL)
- [Phylo.io *Visualising and comparing phylogenetic trees*](https://vdclab-wiki.herokuapp.com/en/visualization/tools-to-make-figures/Phylo-io)
{.links-list}

## Venn Diagram Tools

- [BEG-Venn *An interactive tool for comparing lists with Venn's diagrams*](https://vdclab-wiki.herokuapp.com/en/tools/venn-diagram/BEG-Venn)
- [BioVenn *Comparison and visualization of biological lists using area-proportional Venn diagrams*](https://vdclab-wiki.herokuapp.com/en/tools/venn-diagram/BioVenn)
- [Stefanjol Venny *Online venn diagram generator*](https://vdclab-wiki.herokuapp.com/en/visualization/venn-diagram/Stefanjol-Venny)
- [Venny *An interactive tool for comparing lists with Venn's diagrams*](https://vdclab-wiki.herokuapp.com/tools/venn-diagram/Venny/)
{.links-list}