---
title: Genome-to-Genome Distance Calculator (GGDC)
description: A collection of tools involved in virus/prokaryotic taxonomy, phylogeny, and genome comparison
published: true
date: 2020-10-14T14:44:03.196Z
tags: genomes, taxonomy, phylogeny, virus
editor: markdown
dateCreated: 2020-08-05T14:38:39.167Z
---

# Genome-to-Genome Distance Calculator (GGDC)

> **Genome Sequence-Based (Sub-)Species Delineation.**
> The GGDC web service reports digital DDH for a universal and accurate delineation of prokaryotic (sub-)species without inheriting the pitfalls of classic DDH, and also calculates differences in genomic G+C content.
>
> **Genome-Based Phylogeny and Classification of Phages.**
> Advanced comparison of bacterial and archaeal viruses using their genome or proteome sequences. Results include phylogenomic trees with branch support, suggestions for the classification and publication-ready text.
>
> **Inference of Gene Phylogenies and Gene Similarities.**
> Calculate phylogenetic trees and pairwise gene sequence similarities from 16S rRNA or other genes using state-of-the-art methods.
{.is-info}

 

## Website 

- [GGDC *Home Page*](https://ggdc.dsmz.de/home.php)
- [GGDC *GGDC tool*](https://ggdc.dsmz.de/ggdc.php#)
- [TYGS *TYGS tool*](https://tygs.dsmz.de/)
- [VICTOR *VICTOR tool*](https://ggdc.dsmz.de/victor.php#)
- [Phylogeny Service *Single gene tree tool*](https://ggdc.dsmz.de/phylogeny-service.php#)
 {.links-list}


## Notes from the user
#### GGDC tool, Genome-to-Genome Distance Calculator 2.1
- Only works with NCBI ids.
#### TYGS tool, Type (Strain) Genome Server
- Gives a lot of results to analyse to decide the genealogy of your strains, including full genome alignments and a tree.
#### VICTOR tool, Virus Classification and Tree Building Online Resource
- Results are sent by e-mail.
#### Single gene tree tool
 - Can calculate tree, similiraties or both.
 
## Ease of use
#### GGDC tool
- Every step has a question mark o help you go through it.
- Add the accession numbers of each of your genomes.
- Add our e-mail to be noified of the result.
#### TYGS tool
- Add your genbank or fasta genomes, or add the accession number of your genomes.
- Optionally provide close related species. This is done automaticly. 
- Add our e-mail to be noified of the result.
#### VICTOR tool
- Need to select amino acid or nucleotide (No preselection).
- Enter genebank accession number of your genomes of interest in list, one per line.
- Enter your e-mail to get the result. Results sent in PDF.
#### Single gene tree tool
- Every step has a question mark o help you go through it.
- Add the accession numbers of each of your genomes.
- Add our e-mail to be noified of the result.

## Help
- [General Help *FAQ Page*](https://ggdc.dsmz.de/faq.php#tabPhylogenyService)
 {.links-list}
#### GGDC tool
- [GGDC Help *FAQ Page*](https://ggdc.dsmz.de/faq.php#tabGGDC)
 {.links-list}
#### TYGS tool
- [TYGS Help *FAQ Page*](https://tygs.dsmz.de/faqs)
 {.links-list}
#### VICTOR tool
- [VICTOR Help *FAQ Page*](https://ggdc.dsmz.de/faq.php#tabVICTOR)
 {.links-list}
#### Single gene tree tool
- [Phylogeny Tool Help *FAQ Page*](https://ggdc.dsmz.de/faq.php#tabPhylogenyService)
 {.links-list}
 
## Citation 
#### GGDC tool
- Meier-Kolthoff, J. P., Auch, A. F., Klenk, H. P., & Göker, M. (2013). [Genome sequence-based species delimitation with confidence intervals and improved distance functions.](https://link.springer.com/article/10.1186/1471-2105-14-60) BMC bioinformatics, 14(1), 60.
{.grid-list}
#### TYGS tool
- Meier-Kolthoff, J.P., Göker, M. [TYGS is an automated high-throughput platform for state-of-the-art genome-based taxonomy.](https://www.nature.com/articles/s41467-019-10210-3) Nat Commun 10, 2182 (2019).
{.grid-list}
#### VICTOR tool
- Meier-Kolthoff, J. P., & Göker, M. (2017). [VICTOR: genome-based phylogeny and classification of prokaryotic viruses.](https://academic.oup.com/bioinformatics/article/33/21/3396/3933260) Bioinformatics, 33(21), 3396-3404.
{.grid-list}
