---
title: KEGG PATHWAY Database
description: KEGG PATHWAY is a collection of manually drawn pathway maps representing our knowledge on molecular interaction, reaction and relation networks.
published: true
date: 2020-07-21T21:45:50.892Z
tags: reference, annotation, comparative genomics, data capture, database, metabolism, gene, browser, mapping, metabolic pathways, curated, model
editor: markdown
---

# KEGG PATHWAY Database

> KEGG PATHWAY is a collection of manually drawn pathway maps representing our knowledge on the molecular interaction, reaction and relation networks for: Metabolism, Genetic Information Processing, Environmental Information Processing, Cellular Processes, Organismal Systems, Human Diseases, Drug Development
{.is-info}



## Website

- [KEGG PATHWAY Database *Main Page*](https://www.kegg.jp/kegg/pathway.html)
- [KEGG PATHWAY Help *Pathway Search Help*](https://www.kegg.jp/kegg/document/help_pathway_search.html)
- [KEGG Objects Key *KEGG Object Identifiers*](https://www.kegg.jp/kegg/kegg3.html)
{.links-list}

## Ease of Use

- The KEGG PATHWAY Database is not intuitive to use, but using the "Help" pop-up tabs when provided simplifies the search process.  The collection of KEGG pathways is well organized and easy to browse. 

## User Notes

- KEGG pathways maps contain a wide variety of information.  By clicking the "Help" link on any map Users will see a pop-up window with a key for notations and color coding. 
- KEGG objects are coded by identifiers (see KEGG Objects Key link, above).  It's important that users search for the correct type of object in order to obtain desired results. 

## Citation

- Kanehisa, M. and Sato, Y. (2020) [KEGG Mapper for inferring cellular functions from protein sequences.](https://onlinelibrary.wiley.com/doi/full/10.1002/pro.3711) Protein Sci. 29, 28-35.
{.grid-list}