---
title: PDB-60 Homology Models Database
description: Homology modeling database
published: true
date: 2020-07-29T15:35:28.441Z
tags: database, homology, protein
editor: markdown
---

# PDB-60 Homology Models Database

>  Users can search their sequences for homology models (and structures if they exist) using a wide range of different criteria of importance, such as model quality, coverage, sequence/template e-value or sequence identity, genome, structural template, etc. The database will provide multiple models if they fit the search criteria giving users the opportunity to choose the model most appropriate for their purposes. The user will also have access to coordinate files for the homology models as well as computational function annotations (from the program (MARK-US) for the PDB structure templates). 
>
> If the user is interested in obtaining similar function annotation results for a given model, a link is provide to MARKUS. If a user wishes to obtain a higher quality model, the sequence can be sent to the program PUDGE for refining and/or analyzing the homology model. 
{.is-info}

> **ALL** Searches by amino acid sequence gives a "bad gateway" error code. 
{.is-warning}

## Website 

- [PDB-60 Homology Models Database *Main Page*](http://skybase.c2b2.columbia.edu/pdb60_new/nesg.php)
 {.links-list}

## Ease of Use

- The output of each search is an error message, so this database is not useful. 

## User Notes

- All searches returned error messages. 

## Help

- [PDB-60 Help *Help Page and Tutorial*](http://skybase.c2b2.columbia.edu/pdb60_new/help/help.html)
{.links-list}


## Citation 

- Mirkovic N., Li Z., Parnassa A., Murray D. (2007) [Strategies for High-Throughput Comparative Modeling: Applications to Leverage Analysis in Structural Genomics and Protein Family Organization.](https://onlinelibrary.wiley.com/doi/abs/10.1002/prot.21191) Proteins: Structure, Function, and Bioinformatics 66:766-777.
{.grid-list}