---
title: RaptorX
description: RaptorX is a protein structure prediction server developed by Xu group, excelling at predicting 3D structures for protein sequences without close homologs in the Protein Data Bank (PDB).
published: true
date: 2020-05-26T18:10:46.511Z
tags: tool, sequence analysis, proteins, 2010, prediction, structural alignment, 2009, 2011, 2015, 2012, 2013, model, protein-protein, structural analysis
---

# RaptorX

> RaptorX is a protein structure prediction server developed by Xu group, excelling at predicting 3D structures for protein sequences without close homologs in the Protein Data Bank (PDB). Given an input sequence, RaptorX predicts its secondary and tertiary structures, contacts, solvent accessibility, disordered regions and binding sites. 
&NewLine;
RaptorX also assigns some confidence scores to indicate the quality of a predicted 3D model: P-value for the relative global quality, GDT (global distance test) and uGDT (un-normalized GDT) for the absolute global quality, and modeling error at each residue. 
{.is-info}



## Websites

- [RaptorX *Main Page*](http://raptorx.uchicago.edu/)
- [Help *Documentation*](http://raptorx.uchicago.edu/documentation/)
{.links-list}

## Notes from Users
- Several tools for different prediction purposes:
	1- Regular prediction based on published structures.
  2- De novo property prediction.
  3- Contact prediction.
  4- Protein-protein inteaction prediction
  5- Structural alignment of sequences.
- Possibility to register to save your data.

## Ease of use
- All the tool easy to use, just input your sequence.
- Some results might require some time to interprete.

## Citation

#### Server Paper
- Morten Källberg, Haipeng Wang, Sheng Wang, Jian Peng, Zhiyong Wang, Hui Lu & Jinbo Xu. [Template-based protein structure modeling using the RaptorX web server.](https://www.nature.com/articles/nprot.2012.085) Nature Protocols 7, 1511–1522, 2012.
{.grid-list}

#### Method Papers for tertiary structure prediction
- Ma J, Wang S, Zhao F, Xu J. [Protein threading using context-specific alignment potential. Bioinformatics](https://academic.oup.com/bioinformatics/article/29/13/i257/187813) (Proceedings of ISMB 2013). 2013 Jul 1;29(13):i257-65. doi: 10.1093/bioinformatics/btt210.

- Jianzhu Ma, Jian Peng, Sheng Wang and Jinbo Xu. [A conditional neural fields model for protein threading.](https://academic.oup.com/bioinformatics/article/28/12/i59/268198) Bioinformatics (Proceedings of ISMB 2012), 2012.

- Jian Peng and Jinbo Xu. [RaptorX: exploiting structure information for protein alignment by statistical inference.](https://onlinelibrary.wiley.com/doi/abs/10.1002/prot.23175) PROTEINS, 2011.

- Jian Peng and Jinbo Xu. [A multiple-template approach to protein threading.](https://onlinelibrary.wiley.com/doi/abs/10.1002/prot.23016) PROTEINS, 2011.

- Jian Peng and Jinbo Xu. [Low-homology protein threading.](https://academic.oup.com/bioinformatics/article/26/12/i294/283354) Bioinformatics (Proceedings of ISMB 2010), 2010.

- Jian Peng and Jinbo Xu. [Boosting protein threading accuracy.](https://link.springer.com/chapter/10.1007/978-3-642-02008-7_3) In the Proceedings of the 13th International Conference on Research in Computational Molecular Biology (RECOMB), Lecture Notes in Computer Science. Vol. 5541, pp. 31-45, 2009. Springer.
{.grid-list}

#### Method Paper for secondary structure prediction
- Wang Z, Zhao F, Peng J, Xu J. [Protein 8-class secondary structure prediction using conditional neural fields.](https://onlinelibrary.wiley.com/doi/abs/10.1002/pmic.201100196) Proteomics. 2011 Oct;11(19):3786-92. doi: 10.1002/pmic.201100196. Epub 2011 Aug 31.
{.grid-list}
#### Method Paper for contact prediction
- Zhiyong Wang and Jinbo Xu. [Predicting protein contact map using evolutionary and physical constraints by integer programming.](https://academic.oup.com/bioinformatics/article/29/13/i266/188131) Bioinformatics (Proceedings of ISMB 2013), 2013. Also see an Extended version

- Jianzhu Ma, Sheng Wang, Zhiyong Wang and Jinbo Xu. [Protein Contact Prediction by Integrating Joint Evolutionary Coupling Analysis and Supervised Learning.](https://academic.oup.com/bioinformatics/article/31/21/3506/195691) RECOMB 2015, Lecture Notes in Computer Science , Volume 9029, 2015, pp 218-221.
{.grid-list}
#### Related Papers
- Feng Zhao, Jian Peng and Jinbo Xu. [Fragment-free approach to protein folding using conditional neural fields.](https://academic.oup.com/bioinformatics/article/26/12/i310/283476) Bioinformatics (Proceedings of ISMB 2010), 2010.

- Feng Zhao, Jian Peng, Joe DeBartolo, Karl F. Freed, Tobin R. Sosnick and Jinbo Xu. [A probabilistic and continuous model of protein conformational space for template-free modeling.](https://www.liebertpub.com/doi/abs/10.1089/cmb.2009.0235) Journal of Computational Biology, 2010.
{.grid-list}