---
title: Rfam
description: The Rfam database is a collection of RNA families, each represented by multiple sequence alignments, consensus secondary structures and covariance models (CMs).
published: true
date: 2020-04-08T19:35:45.031Z
tags: sequence alignment, rna, database
---

# Rfam

> The Rfam database is a collection of RNA families, each represented by multiple sequence alignments, consensus secondary structures and covariance models (CMs). 
&NewLine;
The families in Rfam break down into three broad functional classes: non-coding RNA genes, structured cis-regulatory elements and self-splicing RNAs. Typically these functional RNAs often have a conserved secondary structure which may be better preserved than the RNA sequence. The CMs used to describe each family are a slightly more complicated relative of the profile hidden Markov models (HMMs) used by Pfam. CMs can simultaneously model RNA sequence and the structure in an elegant and accurate fashion.
&NewLine;
Rfam families are frequently built from external sources, we ask that if you find a particular family useful for your work that you cite both Rfam and the primary source of our data.
{.is-info}

## Website

- [Rfam *Main Page*](http://rfam.xfam.org/)
{.links-list}

## Citations

- [Rfam 13.0: shifting to a genome-centric resource for non-coding RNA families.](https://academic.oup.com/nar/article/46/D1/D335/4588106) Ioanna Kalvari, Joanna Argasinska, Natalia Quinones-Olvera, Eric P. Nawrocki, Elena Rivas, Sean R. Eddy, Alex Bateman, Robert D. Finn, Anton I. Petrov Nucleic Acids Research (2017)  10.1093/nar/gkx1038
- [Non-coding RNA analysis using the Rfam database.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6754622/) Ioanna Kalvari, Eric P. Nawrocki, Joanna Argasinska, Natalia Quinones-Olvera, Robert D. Finn, Alex Bateman and Anton I. Petrov Current Protocols in Bioinformatics (2018)  10.1002/cpbi.51
{.grid-list}