---
title: Center for non-coding RNA in Technology and Health (RTH)
description: The center aims at developing technologies, computational methods as well as experimental approaches for analysis of the mammalian genome for non-coding RNAs in relation to (inflammatory) diseases.
published: true
date: 2020-06-30T17:46:13.880Z
tags: sequence alignment, rna, resource center, database
editor: markdown
---

# Center for non-coding RNA in Technology and Health (RTH)

> The Center for Non-coding RNA in Technology and Health (RTH) was established on a grant from the Danish Strategic Research Council (Program Committee for Strategic Growth Technologies). Through a multi-disciplinary collaboration, the Center aims at developing technologies, computational methods as well as experimental approaches for analysis of the mammalian genome for non-coding RNAs with respect to (inflammatory) diseases. The Center will focus on developing these technologies and on diabetes.
{.is-info}

 

## Websites

- [RTH *Main Page*](https://rth.dk/)
{.links-list}

### Conserved RNA Structures (CRSs) in the Vertebrate Genome

Data resource and supplement conserved RNA structures in vertebrate genomes.

- [CRSs *Main Page*](https://rth.dk/resources/rnannotator/crs/vert/)
{.links-list}

### Conserved RNA Structures in Drosophila

Conserved RNA structures in insect genomes.

- [Conserved RNA Structures in Drosophila *Main Page*](https://rth.dk/resources/rnannotator/crs/insect/)
{.links-list}

### CRISPRoff

CRISPR-Cas9 off-targeting assessment with nucleic acid duplex energy parameters.

- [CRISPRoff *Main Page*](https://rth.dk/resources/crispr/crisproff/)
{.links-list}

### Foldalign 2.5

Foldalign 2.5: Parallel version of Foldalign for structural alignment of RNA sequences.

- [Foldalign 2.5 *Main Page*](https://rth.dk/resources/foldalign/server/index.html)
{.links-list}

### MaxAlike

Phylogenetic reconstruction of incomplete sequences by maximum likelihood.

- [MaxAlike *Main Page*](https://rth.dk/resources/maxalike/submit.php)
{.links-list}

### mirDBA: microRNA Discovery by Similarity Search to a Database of RNA-seq Profiles

Compare RNA-seq data to a database of pre-compiled miRNA RNA-seq profiles.

- [mirDBA *Main Page*](https://rth.dk/resources/mirdba/)
{.links-list}

### mirnasponge

Predicting clusters of miRNA targets.

- [mirnasponge *Main Page*](https://rth.dk/resources/mirnasponge/)
{.links-list}

### PETfold

Phylogenetic, evolutionary, and thermodynamic folding of a multiple alignment of RNA sequences.

- [PETfold *Main Page*](https://rth.dk/resources/petfold/)
{.links-list}

### RAIN

Searching RNA-protein interactions in STRING integrated framework.

- [RAIN *Main Page*](https://rth.dk/resources/rain/)
{.links-list}

### RILogo

Visualising RNA-RNA interactions.

- [RILogo *Main Page*](https://rth.dk/resources/rilogo/)
{.links-list}

### RNAcop

RNAcop: Optimizing RNA structures by sequence extensions.

- [RNAcop *Main Page*](https://rth.dk/resources/rnacop/)
{.links-list}

### RNAsnp

Identifying local RNA structures disrupted by SNPs.

- [RNAsnp *Main Page*](https://rth.dk/resources/rnasnp/)
{.links-list}

### TFMirLoop

Predicted miRNA and transcription factor loops.

- [TFMirLoop *Main Page*](https://rth.dk/resources/tfmirloop/)
{.links-list}

### WebCircRNA

Assessing the circular RNA potential of coding and noncoding RNA.

- [WebCircRNA *Main Page*](https://rth.dk/resources/webcircrna/)
{.links-list}

## Citations

- [The identification and functional annotation of RNA structures conserved in vertebrates.](https://genome.cshlp.org/content/27/8/1371.short) Seemann SE, Mirza AH, Hansen C, Bang-Berthelsen CH, Garde C, Christensen-Dalsgaard M, Torarinsson E, Yao Z, Workman CT, Pociot F, Nielsen H, Tommerup N, Ruzzo WL, Gorodkin J Genome Res. 2017 Aug;27(8):1371-1383. Epub 2017 May 9
-	[CRISPR-Cas9 off-targeting assessment with nucleic acid duplex energy parameters.](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1534-x) Alkan F, Wenzel A, Anthon C, Havgaard JH, Gorodkin J Genome Biol. 2018 Oct 26;19(1):177
-	[Foldalign 2.5: multithreaded implementation for pairwise structural RNA alignment.](https://academic.oup.com/bioinformatics/article/32/8/1238/1744742) Sundfeld D, Havgaard JH, de Melo AC, Gorodkin J Bioinformatics. 2016 Apr 15;32(8):1238-40. Epub 2015 Dec 24
-	[Identification and characterization of novel conserved RNA structures in Drosophila.](https://link.springer.com/article/10.1186/s12864-018-5234-4) Kirsch R, Seemann SE, Ruzzo WL, Cohen SM, Stadler PF, Gorodkin J BMC Genomics. 2018 Dec 11;19(1):899
-	P. Menzel, P. F. Stadler and J. Gorodkin: [maxAlike: Maximum-likelihood based sequence reconstruction with application to improved primer design for unknown sequences,](https://academic.oup.com/bioinformatics/article/27/3/317/319225) Bioinformatics, 2011, 27, 317-325.
-	P. Menzel, J. Gorodkin and P. F. Stadler: [Maximum Likelihood Estimation of Weight Matrices for Targeted Homology Search,](https://dl.gi.de/handle/20.500.12116/20305) Proceedings of the German Conference on Bioinformatics 2009, LNI. 2009; 211-220
-	Pundhir S and Gorodkin J (2013). [MicroRNA discovery by similarity search to a database of RNA-seq profiles.](https://www.frontiersin.org/articles/10.3389/fgene.2013.00133/full) Front. Genet. 4:133. doi: 10.3389/fgene.2013.00133
-	Langenberger D, Pundhir S, Ekstrøm CT, Stadler PF, Hoffmann S and Gorodkin J.(2012) [deepBlockAlign: a tool for aligning RNA-seq profiles of read block patterns.](https://academic.oup.com/bioinformatics/article/28/1/17/219213) Bioinformatics. 28(1):17-24
-	Seemann SE, Gorodkin J, Backofen R ["Unifying evolutionary and thermodynamic information for RNA folding of multiple alignments"](https://academic.oup.com/nar/article/36/20/6355/2902196), Nucleic Acids Res., 36(20):6355-62, 2008.
-	Seemann SE, Richter AS, Gorodkin J, Backofen R ["Hierarchical folding of multiple sequence alignments for the prediction of structures and RNA-RNA interactions"](https://link.springer.com/article/10.1186/1748-7188-5-22), Algorithms Mol Biol., 5:22, 2010.
-	Seemann SE, Richter AS, Gesell T, Backofen R, Gorodkin J ["PETcofold: Predicting conserved interactions and structures of two multiple alignments of RNA sequences"](https://academic.oup.com/bioinformatics/article/27/2/211/285243), Bioinformatics, 27(2):211-9, 2011.
-	Seemann SE, Menzel P, Backofen R, Gorodkin J ["The PETfold and PETcofold web servers for intra- and intermolecular structures of multiple RNA sequences."](https://academic.oup.com/nar/article/39/suppl_2/W107/2505795), Nucleic Acid Res., 39(Web Server issue):W107-11, 2011.
-	[RAIN: RNA–protein Association and Interaction Networks.](https://academic.oup.com/database/article/doi/10.1093/database/baw167/2884913) Junge A, Refsgaard JC, Garde C, Pan X, Santos A, Alkan F, Anthon C, von Mering C, Workman CT, Jensen LJ, Gorodkin J Database (Oxford). 2017 Jan 10;2017. Print 2017
-	Multiple independent analyses reveal transcription factors are an enriched functional class associated with microRNAs (submission in progress). Larry Croft, Damian Szklarczyk, Lars Juhl Jensen, Jan Gorodkin
-	Xiaoyong Pan, Kai Xiong, et al. [WebCircRNA: Assessing the circular RNA potential of coding and noncoding RNA.](https://www.mdpi.com/2073-4425/9/11/536)
-	Kai Xiong, Xiaoyong Pan, Poul Hyttel, Jan Gorodkin, Kristine K Freude. LIN28A Expresses Circular RNAs in Early Porcine Embryos.
-	Xiaoyong Pan, Kai Xiong. (2015) [PredcircRNA: computational classification of circular RNA from other long non-coding RNA using hybrid features.](https://pubs.rsc.org/en/content/articlelanding/2015/mb/c5mb00214a/unauth#!divAbstract) Mol Biosyst. 2015 Aug;11(8):2219-26
{.grid-list}