---
title: SynMap
description: SynMap generates a syntenic dotplot between two organisms and identifies syntenic regions.
published: true
date: 2020-06-10T15:15:46.863Z
tags: genomes, tool, sequence matching, 2017, data visualization, 2008
editor: markdown
---

# SynMap

> SynMap generates a syntenic dotplot between two organisms and identifies syntenic regions.
Synmap:
> 
> 1) Finds putative genes or regions of homology between two genomes.
> 2) Identies collinear sets of genes or regions of sequence similarity to infer synteny.
> 3) Generates a dotplot of the results and coloring syntenic pairs.
> 
> If you choose, synonymous and non-synonymous site mutation data can be calculated for protein coding genes that are identified as syntenic. These genes will then be colored based on those values in the dotplot for rapid identification of different age-classes of syntenic regions.
{.is-info}


## Website

- [SynMap *Main Page*](https://genomevolution.org/CoGe/SynMap.pl)
{.links-list}

## Notes from Users
- This tool is part of a bigger plateform of tools.
- Some additional analysis options for advanced searches.
- Great to find rearrangments in closely related genomes.

## Ease of use
- Choose your two genomes from the list provide (start by typing).
- Choose if you want to align CDS or genomic DNA.
- Output is a Dotplot of the alignments.

## Citations

- Haug-Baltzell, A., Stephens, S. A., Davey, S., Scheidegger, C. E., & Lyons, E. (2017). [SynMap2 and SynMap3D: web-based whole-genome synteny browsers.](https://academic.oup.com/bioinformatics/article/33/14/2197/3072872) Bioinformatics, 33(14), 2197-2198.
- Lyons, E., Pedersen, B., Kane, J., & Freeling, M. (2008). [The value of nonmodel genomes and an example using SynMap within CoGe to dissect the hexaploidy that predates the rosids.](https://link.springer.com/article/10.1007/s12042-008-9017-y) Tropical Plant Biology, 1(3-4), 181-190.
{.grid-list}