---
title: Tools
description: List of the tools' page that exist on this wiki
published: true
date: 2020-09-03T21:36:47.199Z
tags: category
editor: markdown
---

# Tools

## Alignment tools

- [:star: Dotlet JS *Comparing sequences by the diagonal plot method*](https://vdclab-wiki.herokuapp.com/en/alignment/two-sequences-alignment/Dotlet)
- [Clustal Omega *MSA program that uses seeded guide trees and HMM profile-profile techniques*](https://vdclab-wiki.herokuapp.com/en/alignment/multiple-alignment/EMBL-EBI-MSA-ClustalOmega)
- [:star: EMBL-EBI *Multiple Sequence Alignment*](https://vdclab-wiki.herokuapp.com/en/alignment/multiple-alignment/EMBL-EBI-MSA)
- [:star: GUIDANCE2 *Weighting, filtering or masking unreliably aligned positions in sequence alignments before subsequent analysis*](https://vdclab-wiki.herokuapp.com/en/alignment/editing/GUIDANCE2)
- [LALIGN *Find multiple matching subsegments in two sequences*](https://vdclab-wiki.herokuapp.com/en/alignment/two-sequences-alignment/LALIGN)
- [LAST *LAST finds similar regions between sequences*](https://vdclab-wiki.herokuapp.com/en/alignment/two-sequences-alignment/LAST)
- [:star: MAFFT *Multiple alignment program for amino acid or nucleotide sequences*](https://vdclab-wiki.herokuapp.com/en/alignment/multiple-alignment/MAFFT)
- [:star: MultAlin *MSA from a group of related sequences using progressive pairwise alignments*](https://vdclab-wiki.herokuapp.com/en/alignment/multiple-alignment/MultAlin)
- [:star: MView *Reformats the results of a sequence database search or a multiple alignment*](https://vdclab-wiki.herokuapp.com/en/visualization/sequence-data-visualization/MView)
- [Pairwise Align DNA *Basic DNA pairwise alignment tool*](https://vdclab-wiki.herokuapp.com/en/alignment/two-sequences-alignment/Pairwise-Align-DNA)
- [Pairwise Align Protein *Basic protein pairwise alignment tool*](https://vdclab-wiki.herokuapp.com/en/alignment/two-sequences-alignment/Pairwise-Align-Protein)
- [:star: Pairwise Alignment *Basic protein pairwise alignment tool*](https://vdclab-wiki.herokuapp.com/en/alignment/two-sequences-alignment/Pairwise-Alignment)
- [:star: PRSS *Evaluates the significance of a protein sequence alignment*](https://vdclab-wiki.herokuapp.com/en/alignment/two-sequences-alignment/PRSS)
- [RibEx *Search any sequence for known riboswitches as well as other predicted bacterial regulatory elements*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/RibEx)
- [:star: SIAS *Calculates pairwise sequence identity and similarity from multiple sequence alignments*](https://vdclab-wiki.herokuapp.com/en/tools/alignment/SIAS)
- [:star: T-COFFEE *Align Protein, DNA and RNA sequences*](https://vdclab-wiki.herokuapp.com/en/alignment/multiple-alignment/T-COFFEE)
{.links-list}

## Analysis Tools

- [:star: CD-Search *Searching the Conserved Domain Database with protein or nucleotide query sequences*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/CD-Search)
- [Center for the Study of Systems Biology *Dedicated to the research interests, publications around Jeffrey Skolnick*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/Jeffrey-Skolnick)
- [CMBL MotLoc *Uses aligned set of DNA sequence motifs  and finds similar motifs in the analyzed sequence*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/CMBL-MotLoc)
- [:star: Compute pI *Allows the computation of the theoretical pI (isoelectric point) and Mw (molecular weight)*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/Compute-pI)
- [:star: ENDscript *Extracts and renders a comprehensive analysis of primary to quaternary protein structure information*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/ENDscript)
- [:star: Enzyme Function Initiative (EFI) *Creating and interacting with sequence similarity networks*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/Enzyme-Function-Initiative)
- [:star: ESPript *Program which renders sequence similarities and secondary structure information from aligned sequences*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/ESPript)
- [:star: ETviewer *Relative rank of functional and structural importance among protein homologs sequence positions*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/ETviewer)
- [FungiPath *Orthologous proteins in fungal genomes database*](https://vdclab-wiki.herokuapp.com/databases/data-integration/FUNGIpath/)
- [:star: Genoscope - MicroScope *MicroScope offers many useful data-exploring functionalities*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/Genoscope-Microscope)
- [IDDomainSpotter *Sequence-based approach to assess and visualize domain organisation*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/IDDomainSpotter)
- [:star: MolBiol-Tools *Online tools and resources for molecular biologists*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/MolBiol-Tools)
- [:star: MEME-Suite *Discover novel motifs in collections of unaligned nucleotide or protein sequences*](https://vdclab-wiki.herokuapp.com/en/tools/sequence-logos/The-MEME-Suite)
- [MetNetDB *Information on networks of metabolic and regulatory and interactions in Arabidopsis*](https://vdclab-wiki.herokuapp.com/tools/analysis/MetNetDB)
- [:star: MitoMiner *Database of mammalian mitochondrial localisation evidence, phenotypes, and diseases*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/MitoMiner)
- [:star: NCBI *Conserved Domains*](https://vdclab-wiki.herokuapp.com/tools/analysis/NCBI-Conserved-Domains/)
- [:star: NPS@ *An interactive web server dedicated to protein sequence analysis.*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/NPS@)
- [PDBeMotif *Fast and powerful search tool that facilitates exploration of the Protein Data Bank (PDB)*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/PDBeMotif)
- [:star: Phobius *Prediction of transmembrane topology and signal peptides*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/Phobius)
- [:star: PROSITE *Database of protein families and domains*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/ExPASy-PROSITE)
- [:star: ProteomeHD *Protein Co-regulation Analysis*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/ProteomeHD)
- [ProtFus *Resource that has information about fusion proteins and their interactions*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/ProtFus)
- [:star: PSORTb *Precise bacterial localization prediction tool*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/PSORTb)
- [RDP *Aligned and annotated Bacterial and Archaeal 16S rRNA sequences*](https://vdclab-wiki.herokuapp.com/tools/analysis/RDP/)
- [:star: SignalP *Signal peptide and cleavage sites in gram+, gram- and eukaryotic amino acid sequences*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/SignalP)
- [:star: SoftBerry *Tools for genomic using computational methods of high throughput biomedical data analysis*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/SoftBerry)
- [:star: String *Protein-Protein Interaction Networks and Functional Enrichment Analysis*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/String)
- [:star: SynMap *Generates a syntenic dotplot between two organisms and identifies syntenic regions*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/SynMap)
- [:star: TargetP *Subcellular location of proteins: mitochondrial, chloroplastic, secretory pathway*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/TargetP)
- [TISSUES *Tissue expression database*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/TISSUES)
- [:star: TMHMM *Prediction of transmembrane helices in proteins*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/TMHMM)
- [:star: TopSearch *Gateway to protein structures and their structural relationships*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/TopSearch)
- [ViennaRNA *Several stand-alone programs for the prediction and comparison of RNA secondary structures*](https://vdclab-wiki.herokuapp.com/en/tools/analysis/ViennaRNA)
- [:star: Zhang Lab *Web page dedicated to the research, publications, information of the Zhang Lab*](https://vdclab-wiki.herokuapp.com/tools/analysis/Zhang-Lab)
{.links-list}

## Cloning Tools Platforms

- [Addgene *Global, nonprofit repository that was created to help scientists share plasmids*](https://vdclab-wiki.herokuapp.com/en/tools/cloning-platforms/Addgene)
- [:star: Benchling *Electronic notebook and a suite of sequence design tools for free to academics*](https://vdclab-wiki.herokuapp.com/en/tools/cloning-platforms/Benchling)
- [Serial Cloner *Intuitive interface that assists you in DNA cloning, sequence analysis and visualization*](https://vdclab-wiki.herokuapp.com/en/tools/cloning-platforms/Serial-Cloner)
{.links-list}

## Creating Or Finding Sequence Logos

- [:star: Berkeley WebLogo *Designed to make the generation of sequence logos as easy and painless as possible*](https://vdclab-wiki.herokuapp.com/en/visualization/sequence-data-visualization/Berkeley-WebLogo)
- [:star: Skylign *Creating logos representing both sequence alignments and profile hidden Markov models*](https://vdclab-wiki.herokuapp.com/en/tools/sequence-logos/Skylign)
- [:star: WebLogo3 *Take multiple sequence alignments and generate a graphical representation of the consensus*](https://vdclab-wiki.herokuapp.com/en/tools/sequence-logos/WebLogo3)
{.links-list}

## Database Linker

- [:star: bioDBnet-db2db *Database to Database Conversions*](https://vdclab-wiki.herokuapp.com/en/tools/database-linker/bioDBnet-db2db)
- [BioGPS *Extensible and customizable gene annotation portal*](https://vdclab-wiki.herokuapp.com/en/tools/database-linker/BioGPS)
- [LinkDB *database of links*](https://vdclab-wiki.herokuapp.com/en/tools/database-linker/LinkDB)
- [:star: UniProt-Mapping *Very usuful tool when you want to swap IDs from a database to another*](https://vdclab-wiki.herokuapp.com/en/tools/database-linker/UniProt-Mapping)
{.links-list}

## Data formatting

- [EMBOSS Seqret *Extracting sequences from databases, displaying sequences, reformatting sequences*](https://vdclab-wiki.herokuapp.com/en/tools/data-formatting/EMBOSS-Seqret)
- [Reverse Complement *Converts a DNA sequence into its reverse, complement, or reverse-complement*](https://vdclab-wiki.herokuapp.com/en/tools/data-formatting/Reverse-Complement)
- [:star: Sequence Massager *Easy modification and editing of user-inputted nucleic acid sequences*](https://vdclab-wiki.herokuapp.com/en/tools/data-formatting/Sequence-Massager)
- [XSuLT *Annotation and formatting of a multiple protein sequence alignment or single structure*](https://vdclab-wiki.herokuapp.com/en/tools/data-formatting/XSuLT)
{.links-list}

## Gene finders

- [:star: BLAST *Finds regions of local similarity between sequences*](https://vdclab-wiki.herokuapp.com/en/tools/gene-finders/BLAST)
- [:star: Genevisible *Find the top tissues, cell lines, cancers or perturbations for your gene of interest*](https://vdclab-wiki.herokuapp.com/en/tools/gene-finders/Genevisible)
- [:star: HMMER *Fast and sensitive homology searches*](https://vdclab-wiki.herokuapp.com/tools/gene-finders/EMBL-EBI-HMMER/)
- [:star: InterMine *Integrates biological data sources, making it easy to query and analyse data*](https://vdclab-wiki.herokuapp.com/en/tools/gene-finders/InterMine)
- [ProtPhylo *Web server to retrieve phenotype-to-protein and protein-to-protein functional associations*](https://vdclab-wiki.herokuapp.com/en/tools/gene-finders/ProtPhylo)
{.links-list}

## Gene Neighborhoods

- [FunGeco *Functional potential of bacterial genomes and microbiomes from gene context information*](https://vdclab-wiki.herokuapp.com/en/tools/gene-neighborhoods/FunGeco)
- [Phagonaute *Phagonaute is a new phage genome browser*](https://vdclab-wiki.herokuapp.com/en/tools/gene-neighborhoods/Phagonaute)
{.links-list}

## Gene Networks

- [:star: 3did *3D interacting domains (3did) is a catalog of proteinâ€“protein interactions*](https://vdclab-wiki.herokuapp.com/en/tools/gene-networks/3did)
- [AraNet *Functional gene network for Arabidopsis thaliana*](https://vdclab-wiki.herokuapp.com/en/tools/gene-networks/AraNet)
- [:star: EcoliNet *Probabilistic functional gene network for Escherichia coli*](https://vdclab-wiki.herokuapp.com/en/tools/gene-networks/EcoliNet)
- [:star: GeneMANIA *Predict the functions of your favorite genes and gene sets*](https://vdclab-wiki.herokuapp.com/en/tools/gene-networks/GeneMANIA)
- [GPS-Prot *Data Visualization for Protein-Protein Interactions*](https://vdclab-wiki.herokuapp.com/en/tools/gene-networks/GPS-Prot)
- [:star: IMP *Analyze your experimental results in the functional context of gene-gene networks*](https://vdclab-wiki.herokuapp.com/en/tools/gene-networks/IMP)
{.links-list}

## Gene Ontology Enrichment Tools

- [AgriGO *Web-based tool and database for gene ontology analyses*](https://vdclab-wiki.herokuapp.com/en/ontology/AgriGO)
- [DAVID *Database for Annotation, Visualization and Integrated Discovery*](https://vdclab-wiki.herokuapp.com/en/tools/gene_ontology/DAVID)
- [DeepGOPlus *Predicting protein functions based on protein sequence*](https://vdclab-wiki.herokuapp.com/en/tools/gene_ontology/DeepGOPlus)
- [DiVenn *Venn diagram is widely used to compare gene lists between multiple experiments*](https://vdclab-wiki.herokuapp.com/en/tools/gene_ontology/DiVenn)
- [:star: GO Enrichment Analysis *Representation of our current scientific knowledge about the functions of genes*](https://vdclab-wiki.herokuapp.com/en/tools/gene_ontology/GO-Enrichment-Analysis)
- [GOrilla *Identifying and visualizing enriched GO terms in ranked lists of genes*](https://vdclab-wiki.herokuapp.com/en/tools/gene_ontology/GOrilla)
- [Metascape *A Gene Annotation & Analysis Resource*](https://vdclab-wiki.herokuapp.com/en/tools/gene_ontology/Metascape)
- [PRO *Ontological representation of protein-related entities by explicitly defining them and showing the relationships*](https://vdclab-wiki.herokuapp.com/en/tools/gene_ontology/PROConsortium)
- [The BAR *Explore large-scale data sets from Arabidopsis and other species, and for hypothesis generation*](https://vdclab-wiki.herokuapp.com/en/tools/gene_ontology/The-BAR)
- [YeastEnrichr *New gene list enrichment analysis tool for Saccharomyces cerevisiae*](https://vdclab-wiki.herokuapp.com/en/tools/gene_ontology/YeastEnrichr)
{.links-list}

## Motifs Analysis Tools

- [Domain mapping of disease mutations *Database in which each disease mutation can be displayed by its gene*](https://vdclab-wiki.herokuapp.com/en/tools/motifs-analysis-tools/DMDM)
- [:star: MyHits *Hits is a free database devoted to protein domains*](https://vdclab-wiki.herokuapp.com/en/tools/motifs-analysis-tools/MyHits)
{.links-list}

## Phylogeny Tools

- [:star: Graph Splitting *Phylogenetic analysis method, which can effectively resolve early evolution of protein families*](https://vdclab-wiki.herokuapp.com/en/phylogeny/tools/Graph-Splitting)
- [:star: IQ-TREE *Efficient software for phylogenomic inference*](https://vdclab-wiki.herokuapp.com/en/phylogeny/tools/IQ-TREE)
- [LeBIBI-QBPP *Quick Bioinformatic Phylogeny of Prokaryotes web-server*](https://vdclab-wiki.herokuapp.com/phylogeny/tools/LeBIBI-QBPP/)
- [MEGASoftware *Software for conducting statistical analysis of molecular evolution and for constructing phylogenetic trees*](https://vdclab-wiki.herokuapp.com/en/phylogeny/tools/MEGASoftware)
- [:star: NGPhylogeny *Reconstructing and analysing phylogenetic relationships*](https://vdclab-wiki.herokuapp.com/en/phylogeny/tools/NGPhylogeny)
- [:star: Simple Phylogeny *Provides access to phylogenetic tree generation methods from the ClustalW2*](https://vdclab-wiki.herokuapp.com/en/phylogeny/tools/Simple-Phylogeny)
{.links-list}

## Phylogenetic Distribution Tools

- [:star: AnnoTree *Visualization of genome annotations across large phylogenetic trees*](https://vdclab-wiki.herokuapp.com/en/phylogeny/phylogenetic-distribution/AnnoTree)
- [:star: JGI-IMG/Microbes *Community resource for analysis and annotation of genome*](https://vdclab-wiki.herokuapp.com/en/databases/prokaryote_databases/JGI-IMG)
- [:star: JGI-IMG/Virus *Community resource for analysis and annotation of genome*](https://vdclab-wiki.herokuapp.com/databases/virus_databases/JGI-IMG-VR/)
- [KEGG Ortholog Cluster *Database of ortholog clusters (OCs) based on the whole genome comparison*](https://vdclab-wiki.herokuapp.com/en/databases/ortholog/KEGG-OC)
- [OrthoDB *OrthoDB presents a catalog of orthologous protein-coding genes*](https://vdclab-wiki.herokuapp.com/en/databases/ortholog/OrthoDB)
- [:star: OrthoInspector *inference of orthologous relationships between protein coding-genes*](https://vdclab-wiki.herokuapp.com/phylogeny/phylogenetic-distribution/OrthoInspector/)
{.links-list}

## Prediction

- [:star: AACompIdent *Allows the identification of a protein from its amino acid composition*](https://vdclab-wiki.herokuapp.com/en/tools/prediction/AACompIdent)
- [AtSubP *Arabidopsis Subcellular Localization Prediction Server*](https://vdclab-wiki.herokuapp.com/en/tools/prediction/AtSubP)
- [:star: COGcollator *Web server for analysis of distant relationships between homologous protein families*](https://vdclab-wiki.herokuapp.com/en/tools/prediction/COGcollator)
- [EffectiveDB *Prediction of bacterial protein secretion*](https://vdclab-wiki.herokuapp.com/en/tools/prediction/EffectiveDB)
- [:star: GapMind *Automated annotation of Amino acid biosynthesis*](https://vdclab-wiki.herokuapp.com/en/tools/prediction/GapMind)
- [:star: Genome3D *Provides consensus structural annotations and 3D models for sequences*](https://vdclab-wiki.herokuapp.com/en/tools/prediction/Genome3D)
- [miRPD *Explicitly inferring miRNA-protein-disease associations*](https://vdclab-wiki.herokuapp.com/en/tools/prediction/miRPD)
- [MMTK *Open Source program library for molecular simulation applications*](https://vdclab-wiki.herokuapp.com/en/tools/prediction/MMTK)
- [ProtParam *Allows the computation of various physical and chemical parameters for a given protein*](https://vdclab-wiki.herokuapp.com/en/tools/prediction/ProtParam)
- [:star: RNAmmer *Predicts 5s/8s, 16s/18s, and 23s/28s ribosomal RNA in full genome sequences*](https://vdclab-wiki.herokuapp.com/en/tools/prediction/RNAmmer)
- [RNApredator *Bacterial sRNA target prediction webserver*](https://vdclab-wiki.herokuapp.com/en/tools/prediction/RNApredator)
- [TRED *Collects cis - and trans -regulatory elements and is dedicated to easy data access and analysis*](https://vdclab-wiki.herokuapp.com/en/tools/prediction/TRED)
- [:star: tRNAscan-SE *Searching for tRNA genes in genomic sequence*](https://vdclab-wiki.herokuapp.com/en/tools/prediction/tRNAscan-SE)
{.links-list}

## Tools to Make Figures

- [Caleydo *Tools that focuses on biological data visualization*](https://vdclab-wiki.herokuapp.com/en/visualization/data-visualization/Caleydo)
- [:star: Clustergrammar *Analyzing high-dimensional data as interactive hierarchically clustered heatmaps*](https://vdclab-wiki.herokuapp.com/en/visualization/data-visualization/Clustergrammar)
- [ETE-Toolkit *A Python framework for the analysis and visualization of trees*](https://vdclab-wiki.herokuapp.com/en/visualization/tools-to-make-figures/ETE-Toolkit)
- [:star: EvolView *Easy-to-use online tool for displaying, managing and customizing phylogenetic trees*](https://vdclab-wiki.herokuapp.com/en/visualization/tools-to-make-figures/EvolView)
- [GraPhlAn *Tool that produces high-quality, compact visualizations of microbial genomes*](https://vdclab-wiki.herokuapp.com/en/phylogeny/tools/GraPhlAn)
- [:star: Heatmapper *Online application for visualizing various types of data as heat maps*](https://vdclab-wiki.herokuapp.com/en/visualization/data-visualization/Heatmapper)
- [:star: iTOL *Online tool for the display, annotation and management of phylogenetic trees*](https://vdclab-wiki.herokuapp.com/en/visualization/tools-to-make-figures/iTOL)
- [PCAGO *Analyzing your RNA-Seq read counts with principal component analysis (PCA)*](https://vdclab-wiki.herokuapp.com/en/visualization/data-visualization/PCAGO)
- [:star: Phylo.io *Visualising and comparing phylogenetic trees*](https://vdclab-wiki.herokuapp.com/en/visualization/tools-to-make-figures/Phylo-io)
{.links-list}

## Visualizing Pathways - No Data Mapping

- [iPath *Web-based tool for the visualization, analysis and customization of various pathway maps*](https://vdclab-wiki.herokuapp.com/en/tools/visualizing-pathways/iPath)
{.links-list}

## Visualizing Pathways - Mapping Data To Pathways

- [ConsensusPathDB *Integrates interaction networks in Homo sapiens*](https://vdclab-wiki.herokuapp.com/en/tools/pathways/ConsensusPathDB)
- [GENIES *Predict potential functional associations between genes*](https://vdclab-wiki.herokuapp.com/en/tools/pathways/GENIES)
- [KEGG-Mapper *Collection of tools for KEGG mapping*](https://vdclab-wiki.herokuapp.com/en/tools/pathways/KEGG-Mapper)
- [:star: PaintOmics *Integrative visualization of multiple omic datasets onto KEGG pathways*](https://vdclab-wiki.herokuapp.com/en/tools/pathways/PaintOmics)
- [:star: PathView *Provides easy interactive access, and generates high quality, hyperlinked graphs*](https://vdclab-wiki.herokuapp.com/en/tools/pathways/PathView)
- [Pathway-Commons *Access and discover data integrated from public pathway and interactions databases*](https://vdclab-wiki.herokuapp.com/en/tools/pathways/Pathway-Commons)
{.links-list}
