---
title:  KEGG Annotation
description: KO-based annotation for linking genomes to phenotypes.
published: true
date: 2020-05-14T12:33:25.752Z
tags: annotation, genomes, metagenomes
---

# KEGG Annotation

> The KEGG database contains three main components for genome/metagenome annotation:
&NewLine;
	• the collection of internally annotated gene catalogs for the complete genomes (called KEGG organisms) and additional protein sequences in the KEGG GENES database
	• the knowledge base of high-level functions represented as the molecular interaction, reaction and relation networks in the KEGG PATHWAY, BRITE and MODULE databases, and
	• the knowledge base of molecular-level functions associated with ortholog groups in the KO database, where most KO entries are defined in a context-dependent manner as nodes of the KEGG molecular networks.
&NewLine;
In general, KO entries (identified by K numbers) also represent sequence similarity groups. Thus, the sequence similarity search of a query genome against KEGG GENES is a search for most appropriate K numbers, and the assigned set of K numbers can be used to reconstruct KEGG pathway maps, BRITE hierarchies and KEGG modules, enabling interpretation of high-level functions
{.is-info}

## Website

- [KEGG Annotation *Main Page*](https://www.genome.jp/kegg/annotation/)
{.links-list}

## User Notes
- The tool to look a the distribution of lists of KO family is very useful.

## Citation

- Kanehisa, M., Sato, Y., Kawashima, M., Furumichi, M., & Tanabe, M. (2016). [KEGG as a reference resource for gene and protein annotation.](https://academic.oup.com/nar/article/44/D1/D457/2502600) Nucleic acids research, 44(D1), D457-D462.
{.grid-list}