---
title: JPred
description: A Protein Secondary Structure Prediction Server.
published: true
date: 2020-07-22T20:54:02.924Z
tags: tool, homology, prediction, protein, structural alignment, model, structural analysis, webserver
editor: markdown
---

# JPred 4

> JPred is a web server that takes a protein sequence or multiple alignment of protein sequences, and from these predicts the location of secondary structures using a neural network called Jnet. The prediction classifies each amino acid residue as belonging to alpha helix ('H'), beta sheet ('E') or not H or E ('-') secondary structures.
&NewLine;
JPred also performs predictions of solved accessibility with Jnet and coiled-coil regions via MultiCoil.
{.is-info}

 

## Website 

- [JPred 4 *Main Page*](http://www.compbio.dundee.ac.uk/jpred4/index.html)
 {.links-list}

## Notes from User
- Easy to use web interface
- Provides link to PDB if structure is available
- Prediction output easy to understand
- Acronyms and visual explained in link on prediction page

## Citation 

- Drozdetskiy, A., Cole, C., Procter, J., & Barton, G. J. (2015). [JPred4: a protein secondary structure prediction server.](https://academic.oup.com/nar/article/43/W1/W389/2467870) Nucleic acids research, 43(W1), W389-W394.
{.grid-list}