---
title: Class Microbial Comparative Genomics MCB6318 
description: Overview of the tools and websites covered in the "Microbial Comparative Genomics" class (MCB6318)
published: true
date: 2023-12-17T20:21:04.401Z
tags: recommended, favorites, workflow, teaching
editor: markdown
dateCreated: 2023-03-01T20:05:36.379Z
---

# MCB6318 - Comparative Microbial Genomics
> This class gives basic tools to capture knowledge on microbial protein families and reviews intergrative datamining methods that allow to predict the function of proteins.  
{.is-info}

### Module 1 - Dealing with the Avalanche of Genomic Data
> With the genomic revolution started in 1995, microbiology has become a data science as all aspects of the field are driven by kwowledge derived from sequences ( see extra reading [here](MCB6318_M1_additional_reading)). With >400,000 genomes sequenced ( see current number on the [GOLD database](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/JGI-GOLD)), gathering informations on genomes and proteins is not easy.
{.is-info}

**Module Objectives**
- Extract and analyze bacterial genomes of interest 
- Extract sets of proteins/genes sequences of a specific family
- Find the literature associated with a protein family
- Map accession numbers from one database to another
- Reformat Fasta files using REGEX  tools

#### Extracting and Analyzing Bacterial Genomes

> Many integrated databases are available to identify or gather genomic bacterial or archaeal sequences. We use BV-BRC routinely but IMG/JGI, Microscope and NCBI are also great resources.
{.is-info}

- [**BV_BRC**](/databases/bacterial_databases/patric)
- [**IMG/JGI**](/databases/prokaryote_databases/JGI-IMG)
- [**NCBI Datasets**](https://www.ncbi.nlm.nih.gov/datasets/)
- [**Microscope**](/tools/analysis/Genoscope-Microscope)
### Notes 
Some genomes might be not correctly classified. Focusing on reference and/or complete genomes is always better when you are gathering genomes. 

#### Extracting and Analyzing eukaryotic & plants Genomes
> For comparative purposes knowing useful fungal or plant databases is sometimes very useful  .
{.is-info}
- [**SGD** *Baker's yeast*](/databases/organism_specific/SGD)
- [**PomBase** *Fission yeast*](/databases/organism_specific/PomBase)
- [**Plaza** *Plants*](/platform/plant-comparative-genomic-tools/PLAZA)
- [**EupathDB** *Eukaryotic MIcrobes*](/databases/data-integration/EuPathDB)


#### Extracting Sets of Protein/Gene Sequences for a Target Family
> Using [BlastP](https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastp&PAGE_TYPE=BlastSearch&LINK_LOC=blasthome) @[NCBI](https://www.ncbi.nlm.nih.gov/) is not the most efficient way to gather diverse set of sequences because you will have too many very similar sequences unless you limit greatly your genomes or your database. [HHmer](https://www.ebi.ac.uk/Tools/hmmer) can help do sensitive searches in a more compact way.  Out favorite methods to gather sequences from a protein family quickly are [orthology databases](resources/general_resources/Quest-for-Orthologs) or [Protein Family databases](https://vdclab-wiki.herokuapp.com/t/protein%20family). One must still be conscious of paralogs and fusions. Uniprot is a great starting point as it federates many other resources. The ones we use frequently are listed below.
{.is-info}

**Orthology databases**
- [**Uniprot**](/UniProtKB-Advanced-Search)
- [**OrthoInspector**](/phylogeny/phylogenetic-distribution/OrthoInspector)
- [**OMA**](https://omabrowser.org/oma/home/)
- [**KEGG KO**](/databases/ortholog/KEGG-OC)

**Protein family databases**
- [**EggNog**](http://eggnog6.embl.de)
- [**InterPro**](/databases/general_databases/InterPro)
- [**CATH**](/databases/data-integration/CATH)
- [**AnnoTree**](/phylogeny/phylogenetic-distribution/AnnoTree)

#### Finding Literature Pertaining to a Target Protein Family

> Finding the papers published on different members of the same protein family can be quite difficult as explained in detail in this manuscript [Reed et al](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10187207/). The main message is to always start by Paper Blast with a Family input (HMM) and then also with phylogenetically diverse sequences.  
{.is-info}

- [**PaperBlast** *Single Sequence input*](/literature-search/PaperBLAST)
- [**PaperBlast** *Family Input*](https://papers.genomics.lbl.gov/cgi-bin/hmmSearch.cgi)
- see also [**VDC Litterature links**](https://vdclab-wiki.herokuapp.com/e/en/literature-search) 

#### Mapping Accessions/Identifiers Between Different Source Databases
> Mapping Ids from one database format to the other is a constant issue when navigating database. Mappers are available but their use is  can be tricky and fustrating.
{.is-info}

- [**DAVID**](https://david.ncifcrf.gov/conversion.jsp)
- [**UniProt Mapper**](https://www.uniprot.org/id-mapping)
- [**BV-BRC Mapper**](https://www.bv-brc.org/app/IDMapper)
- [**KEGG Database mapping**](https://www.genome.jp/linkdb/)
- [**BridgeDB**](https://www.bridgedb.org/)

#### Reformatting FASTA Files Using REGEX Tools
> To do multiple alignments and phylogenetic trees, the headers in Fasta sequences will often need to be reformated. A user with little programming experience can use regular expression manipulations (RegEx) to do that pretty easily.
{.is-info}

[**RegExr**](https://regexr.com)

### Notes 
Registration of a RegExr user account is optimal, as this allows for saving and reuse of script templates to transform or convert different types of sequence headers. Resource also includes a reference and instructional materials for text expression rules, practices

------------------------------------------------------------------------

### Module 2 - Linking Gene and Function
> In the genomic era, protein function is predicted using automated methods during the genome annoation process ( see review by [Stein](https://pubmed.ncbi.nlm.nih.gov/11433356/) that is still very current). Many different types of errors can  occur . See List [here](https://vdclab-wiki.herokuapp.com/en/classes/AnnotationErrors). One of the ways to check for errors is to put the annotation in biological context by performing metabolic reconstructions. It is also a path to identify missing genes or pathway holes. Comparative genomic methods such as physical clustering  or gene neighborhood analyses methods can then identify candidates for these missing genes ( see Review [here](https://pubmed.ncbi.nlm.nih.gov/20001958/)) and - [Additional Reading here](https://vdclab-wiki.herokuapp.com/en/classes/MCB6318_Module2_Additional_reading). Automation of functional anntotation requires controled vocabularies that describes functions such as Gene Ontologies (GO) and/or Enzyme Commission (EC) numbers as described [here](https://vdclab-wiki.herokuapp.com/en/classes/ControlledVocabularies). 
{.is-info}

**Module objectives**
- Place an annotated gene in a biological context
- Check if a metabolic pathway is complete in a sequenced organism
- Visualize and compare pathway information in different databases
- Identify pathway holes
- Conduct phylogenetic distribution queries
- Analyze physical clustering

#### Exploring Pathway databases
> Databases that compile patwhay information and place predicted proteins in specific pathways starting with genomic information can be used to perform metabolic reconstruction, check if a pathway is complete and compare pathways between organisms. KEGG has both Pathways and Modules even if the latter have been also intergrated in the former. Metacyc/BioCyc uses now a subscription model. 
{.is-info}
- [**KEGG Pathways**](https://vdclab-wiki.herokuapp.com/en/databases/enzyme-pathway/KEGG-PATHWAY)
- [**KEGG Modules**](https://www.genome.jp/kegg/module.html)
- [**MetaCyc/BioCyc**](https://vdclab-wiki.herokuapp.com/e/en/databases/enzyme-pathway/MetaCyc)

> General genome integration databases also have tools to perform metabolic reconstructions and they have nice features to compare pathways between genomes. 
{.is-info}
- [**IMG**](https://img.jgi.doe.gov/cgi-bin/m/main.cgi)
- [**BV-BRC**](https://www.bv-brc.org/app/ComparativeSystems)
- [**Microscope**]((https://mage.genoscope.cns.fr/microscope/metabolism/index.php?))

#### Finding candidates for "missing genes" or pathway holes
> Once you have identified a globally or missing gene case  by doing metabolic reconstruction, or through literature searches or looking at the [Unknown Gene/Enzyme Databases](https://vdclab-wiki.herokuapp.com/en/databases),  you can use different types of genomic or post-genomic derived associations to  find gene candidates. In this module we focus on protein fusions,  physical clustering and phylogenetic profiles, in the next modules we will look at co-regulation, protein interactions  and omics derived associations.All metabolic pathways have pathway holes and non-orthologous displacments that fill these holes. [See this analysis of the Folate pathway](https://pubmed.ncbi.nlm.nih.gov/25210598/).
{.is-info}

>Integrative database. The most user friendly database to integrate differnt types of association evidence is STRING. It is a good strating point but as everything is pre-computed. PubSEED has a great physical clustering tool but the genomes have not been update in many years. This tools is now integrated in [BV-BRC](/databases/bacterial_databases/patric).
{.is-info}
- [**String**](https://vdclab-wiki.herokuapp.com/en/tools/analysis/String)
- [**PubSEED**](https://vdclab-wiki.herokuapp.com/en/databases/bacterial_databases/pubseed)
- [**MBGD**](/databases/ortholog/MBGD)

> Additional gene clustering platform. There are many tools to explore physical clusteringa associations. The ones we use the most in addition to intergartive databases.
{.is-info}

- [**Fastgenomics** *very quick fisrt overview but only 6000 genomes*](https://fast.genomics.lbl.gov/cgi/search.cgi)
- [**GeCoViz** *nice graphics based o EggNog*](https://gecoviz.cgmlab.org)
- [**EFI GNN** *Most extensive in terms of data, all Uniprot*](https://efi.igb.illinois.edu/efi-gnt/)
- [**ProFaNA**](http://bioinfo.sggw.edu.pl/neighborhood_analysis/)

> Fusion databases. It is difficult to differentiate between true fusion (or Rosetta stone proteins) from multifomain proteins.  The former have to be found as seperate domains in some organisms.  Model SEED  is a specific  fusion database but otherwise general domain database have indirectly fusion information.
{.is-info}

- [**CDD @NCBI**](https://www.ncbi.nlm.nih.gov/Structure/lexington/lexington.cgi)
- [**InterPRO**](https://vdclab-wiki.herokuapp.com/en/databases/general_databases/InterPro)
- [**EggNog**](http://eggnog6.embl.de)
- [**Model SEED Fusion**](https://modelseed.org/projects/fusions/)

> Phylogenetic Distribution. The tools to  extract  the  protein family are present and absent in specific  are intergarted in the general integrative genome databases. THye vary by the number and taxonomy of genomes. The difficulty lies in choosing enough genomes to  have a reasonable of candidates. 
{.is-info}

- [**Microscope Phyloprofiler**](https://mage.genoscope.cns.fr/microscope/compgenomics/phyloprofil.php)
- [**MicrobesOnline Phyloprofiler**](http://www.microbesonline.org/cgi-bin/matchphyloprofile.cgi)
- [**JGI/IMG Phyloprofiler**](https://img.jgi.doe.gov/cgi-bin/m/main.cgi?section=PhylogenProfiler&page=phyloProfileForm)
- [**OrthoMCL**](https://orthomcl.org/orthomcl/app/#phyleticPattern)


#### AI tools to predict protein functions
> Give background on AI TO DO 
{.is-info}

- [**Predicting EC number from sequence**](https://clean.platform.moleculemaker.org/configuration)
- [**Predicting Gene Ontology from sequence**](https://google-research.github.io/proteinfer/)


---
### Module 3 - Genome-wide Analysis of Experimental Data and Data Integration
> In the last 20 years different types of "omics" data  (transcriptomics, proteomics, metabolomics to name the most common) can be generated and the amount is increasing at an exponential rate as the costs go down and the technology improves. 
In parallel predictions of regulatory network based on motif analyses has improved and when combined with transcriptomic data can help identify genes that are co-regulated. One problem with all these data sets is that they are noisy and require replicates ( from 3 to 6 depending on the techniques).  Also, Id mapping issues make the use of web-based analysis tools sometimes fustrating. Many of the best tools to analyze multi-omics data are R packages but there are few web-based alternatives to both analyze your own and published data. In front of this data deluge, a useful to keep  Track is [OmicsDI](https://www.omicsdi.org/). See addional reading Here to do
{.is-info}

**Objective Summary**
- Describe the overall process used to generate and analyze gene expression data
- Extract and analyze gene expression data from databases
- Predict and evaluate the presence of DNA binding sites for specific transcription factors
- Analyze Phenotype/Fitness data 
- Map omics data to pathways

#### Gene Expression Data: Generation, Extraction, & Analysis
>  Transcriptomics data was initially captured using Microarrays and is now mainly generated using RNAseq. 

**Platforms to analyze your microbial RNASeq data ( Free and web-based)
- Kbase
- BV-BRC
- Microscope 

**Multi Species Depositories and/or analyses**
- [**GEO2R** *All transcriptomic data must be deposited in GEO before publication*](/databases/microarray-rnaseq/NCBI-GEO)
- [**OmicsDI** *Compilation of all types of omics data*](https://www.omicsdi.org/)
- [**Atted-II** *Plant expression data*](/databases/microarray-rnaseq/ATTED-II)
- [**CoExpressDB** * Model Eukaryotes expression data*](https://coxpresdb.jp/)
- [**iModulon**](/omics-data/omics-data/iModulon)

**Organisms Specific databases**

- [**GeneXP** *E. coli expression data*](https://genexpdb.okstate.edu/)
- [**PneumoExpress** *Streptococcus pneumoniae expression data*](https://vdclab-wiki.herokuapp.com/en/omics-data/omics-data/PneumoExpress)
- [**SPELL** *S. cerevisiae expression  data*](/databases/organism_specific/SPELL)	

#### Transcription Factor Binding Sites: Prediction & Evaluation

#### Analyzing Phenotype & Fitness Data
> Ordered mutant libaries have been constructed for a dozen model organisms. This allowed to identify essential genes  and screen these mutants in a wide variety of growth conditions including the presence of many chemicals. More recently TnSeq TnSeq-derived fitness data or [CRISPRi knockout data](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8510551/) can be generated more quickly. Accessing the phenome data can be sometimes difficult (for example the data from the *E.coli* Carol Gross [data set](https://pubmed.ncbi.nlm.nih.gov/21185072/) is not longer available)
{.is-info}

- [**OGEE** *essentiality data*](databases/phenotypes-fitness/OGEE)
- [**DEG** *essentiality data*](/databases/data-integration/DEG)
- [**FitnessBrowser** *TnSeq Data*](databases/phenotypes-fitness/FitnessBrowser)
- [**FitDB** *S. cerevisiae Phenome data*](/databases/phenotypes-fitness/FitDB)
- [**Yeast Phenome** *S. cerevisiae Phenome data*](https://www.yeastphenome.org/)



#### Data Integration and Mapping Omics Data to Pathways

---
### Module 4 - Mining and Predicting 3D Protein Structures

(insert diagram of workflow)
**Objective Summanry**
- Visualize 3D protein structures
- Build models of protein structures
- Identify ligand-binding pockets
- Predict Protein-Protein or Protein-Nucleic acid binding interfaces

#### Visualizing 3D Protein Structures

#### Building Models of Protein Structures

#### Identifying Ligand-binding Sites

#### Predicting Protein-Protein, Protein-NA Binding Interfaces


-[](https://inga.bio.unipd.it/)

---
### Module 5 - Data Visualization ***(more specific?)***

(insert diagram of workflow)
**Objective Summary**
- Discuss different network visualization tools
- Visualize gene neighborhoods
- Compare sequence Logos
- Add different types of data to phylogenetic trees

#### Network Visualization Tools

#### Gene Neighborhood Analysis

#### Comparing Sequence Logos

#### Mapping Data to Phylogenetic Trees (Phylogenomics)
