---
title: iTOL: Interactive Tree of Life
description: Interactive Tree Of Life is an online tool for the display, annotation and management of phylogenetic trees.
published: true
date: 2020-10-01T13:13:02.780Z
tags: annotation tools, taxonomy, phylogeny, data visualization
editor: markdown
dateCreated: 2020-04-16T16:47:13.561Z
---

# iTOL: Interactive Tree of Life

> Interactive Tree Of Life is an online tool for the display, annotation and management of phylogenetic trees. Explore your trees directly in the browser, and annotate them with various types of data. All datasets can be be displayed simultaneously, with fine-grained interactive control of their position, size and other parameters.
>
> iTOL provides WYSIWYG high quality export into vector and bitmap graphics formats. Interactively adjust the tree as you like, and the exported figure will mirror what is shown on your screen.
{.is-info}

> 1st of October 2020:: iTOL is not free anymore, even if there is still a free access mode with this restriction:
— Trees uploaded anonymously (i.e. without an iTOL account) are removed after 48 hours.
— Tree annotations can be added normally, but cannot be saved. After completing the annotation, users have to export their figures right away.
— Batch upload mode is not available.
— Existing trees in user's account can be viewed and exported without restrictions, but no new annotations, parameter changes or views can be saved.
— User account can be used normally, with unlimited creation of workspaces and projects. New trees can be uploaded into user projects as usual, but any new tree annotations, views or parameters cannot be saved.
{.is-warning}


## Website

- [iTOL *Main Page*](https://itol.embl.de/)
{.links-list}

## Notes from the user
- A great tool for visualizing phylogenetic trees
- The user must already have a phylogenetic tree file (Newick, Nexus or PhyloXML)
- Offers a wide variety of ways to annotate your trees with metadata
- Can export vector images of generated trees
- One major issue is that iTOL requires a newick file  with Taxonomic ids as input for spceis trees. The associated [PhyloT tool](https://phylot.biobyte.de) was great but it is not free for over >10 genomes. So Work-arounds using Patric can be done
- There now also  are adding paying features to iTOL itself, so for the moment, it is still the most user friendly tool to map data to trees but if they are not free , it might be worth switching to  [ETE](https://vdclab-wiki.herokuapp.com/visualization/tools-to-make-figures/ETE-Toolkit) or [Evolview](https://vdclab-wiki.herokuapp.com/EvolView)

## Ease of use

iTOL is a simple web-tool that allows the user to completely customize their own phylogenetic trees. The interface is easy to use and understand. The user must use a second service or software to generate a phylogenetic tree before iTOL can be used. 

## Citations

- Letunic I and Bork P (2006) Bioinformatics 23(1):127-8 [Interactive Tree Of Life (iTOL): an online tool for phylogenetic tree display and annotation](https://academic.oup.com/bioinformatics/article/23/1/127/188940)
- Letunic I and Bork P (2011) Nucleic Acids Res doi: 10.1093/nar/gkr201 [Interactive Tree Of Life v2: online annotation and display of phylogenetic trees made easy](https://academic.oup.com/nar/article/39/suppl_2/W475/2505767)
-	Letunic I and Bork P (2016) Nucleic Acids Res doi: 10.1093/nar/gkw290 [Interactive Tree Of Life (iTOL) v3: an online tool for the display and annotation of phylogenetic and other trees](https://academic.oup.com/nar/article/44/W1/W242/2499315)
-	Letunic I and Bork P (2019) Nucleic Acids Res doi: 10.1093/nar/gkz239 [Interactive Tree Of Life (iTOL) v4: recent updates and new developments](https://academic.oup.com/nar/article/47/W1/W256/5424068)
{.grid-list}