---
title: MCB6318 - Recommended Workflows
description: Recommended pipelines designed to emulate the formalized advanced course, MCB6318.
published: true
date: 2023-12-16T04:04:24.548Z
tags: recommended, favorites, workflows, workflow
editor: markdown
dateCreated: 2023-03-01T20:05:36.379Z
---

# MCB6318 - Comparative Microbial Genomics
> This class gives basic tools to capture knowledge on microbial protein families and reviews intergrative datamining methods that allow to predict the function of proteins.  
{.is-info}

### Module 1 - Dealing with the Avalanche of Genomic Data
**Objective Summary**
- Extract and analyze bacterial genomes of interest 
- Extract sets of proteins/genes sequences of a specific family
- Find the literature associated with a protein family
- Map accession numbers from one database to another
- Reformat Fasta files using REGEX  tools

#### Extracting and Analyzing Bacterial Genomes

> Many integrated databases are available to identify or gather genomic bacterial or archaeal sequences. We use BV-BRC routinely but IMG/JGI, Microscope and NCBI are also great resources.
{.is-info}

- [**BV_BRC**](/databases/bacterial_databases/patric)
- [**IMG/JGI**](/databases/prokaryote_databases/JGI-IMG)
- [**NCBI Datasets**](https://www.ncbi.nlm.nih.gov/datasets/)
- [**Microscope**](/tools/analysis/Genoscope-Microscope)
### Notes 
Some genomes might be not correctly classified. Focusing on reference and/or complete genomes is always better when you are gathering genomes. 

#### Extracting and Analyzing eukaryotic & plants Genomes
> For comparative purposes knowing useful fungal or plant databases is sometimes very useful  .
{.is-info}
- [**SGD** *Baker's yeast*](/databases/organism_specific/SGD)
- [**PomBase** *Fission yeast*](/databases/organism_specific/PomBase)
- [**Plaza** *Plants*](/platform/plant-comparative-genomic-tools/PLAZA)
- [**EupathDB** *Eukaryotic MIcrobes*](/databases/data-integration/EuPathDB)


#### Extracting Sets of Protein/Gene Sequences for a Target Family
> Using [BlastP](https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastp&PAGE_TYPE=BlastSearch&LINK_LOC=blasthome) @[NCBI](https://www.ncbi.nlm.nih.gov/) is not the most efficient way to gather diverse set of sequences because you will have too many very similar sequences unless you limit greatly your genomes or your database. [HHmer](https://www.ebi.ac.uk/Tools/hmmer) can help do sensitive searches in a more compact way.  Out favorite methods to gather sequences from a protein family quickly are [orthology databases](resources/general_resources/Quest-for-Orthologs) or [Protein Family databases](https://vdclab-wiki.herokuapp.com/t/protein%20family). One must still be conscious of paralogs and fusions. Uniprot is a great starting point as it federates many other resources. The ones we use frequently are listed below.
{.is-info}

**Orthology databases**
- [**Uniprot**](/UniProtKB-Advanced-Search)
- [**OrthoInspector**](/phylogeny/phylogenetic-distribution/OrthoInspector)
- [**OMA**](https://omabrowser.org/oma/home/)
- [**KEGG KO**](/databases/ortholog/KEGG-OC)

**Protein family databases**
- [**EggNog**](http://eggnog6.embl.de)
- [**InterPro**](/databases/general_databases/InterPro)
- [**CATH**](/databases/data-integration/CATH)'
- [**AnnoTree**](/phylogeny/phylogenetic-distribution/AnnoTree)

#### Finding Literature Pertaining to a Target Protein Family
(see broader [Literature Capture Workflow](link))
> Finding the papers published on different members of the same protein family can be quite difficult as explained in detail in this manuscript [Reed et al](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10187207/). The main message is to always start by Paper Blast with a Family input (HMM) and then also with phylogenetically diverse sequences.  
{.is-info}

- [**PaperBlast** *Single Sequence input*](/literature-search/PaperBLAST)
- [**PaperBlast** *Family Input*](https://papers.genomics.lbl.gov/cgi-bin/hmmSearch.cgi)

#### Mapping Accessions/Identifiers Between Different Source Databases
> Mapping Ids from one database format to the other is a constant issue when navigating database. Mappers are available but their use is  can be tricky and fustrating.
{.is-info}

- [**DAVID**](https://david.ncifcrf.gov/conversion.jsp)
- [**UniProt Mapper**](https://www.uniprot.org/id-mapping)
- [**BV-BRC Mapper**](https://www.bv-brc.org/app/IDMapper)
- [**KEGG Database mapping**](https://www.genome.jp/linkdb/)

#### Reformatting FASTA Files Using REGEX Tools
> To do multiple alignments and phylogenetic trees, the headers in Fasta sequences will often need to be reformated. A user with little programming experience can use regular expression manipulations (RegEx) to do that pretty easily.
{.is-info}

[**RegExr**](https://regexr.com)

### Notes 
Registration of a RegExr user account is optimal, as this allows for saving and reuse of script templates to transform or convert different types of sequence headers. Resource also includes a reference and instructional materials for text expression rules, practices

---

### Module 2 - Linking Gene and Function
(insert diagram of workflow)
**Objective Summary**
- Place an annotated gene in a biological context
- Check if a metabolic pathway is complete in a sequenced organism
- Visualize and compare pathway information in different databases
- Identify pathway holes
- Conduct phylogenetic distribution queries
- Analyze physical clustering

#### Placing Annotated Genes in Biological Context

#### Checking Metabolic Pathways for Completeness in Sequenced Organisms

#### Visualize and Compare Pathway Information Between Databases

#### Identifying Pathway Holes, Overlap, and Redundancy
(embed phylogenetic distribution, physical clustering strategies within this subsection?)

---
### Module 3 - Genome-wide Analysis of Experimental Data and Data Integration

(insert diagram of workflow)
**Objective Summary**
- Describe the overall process used to generate and analyze gene expression data
- Extract and analyze gene expression data from databases
- Predict and evaluate the presence of DNA binding sites for specific transcription factors
- Analyze Phenotype/Fitness data
- Map omics data to pathways

#### Gene Expression Data: Generation, Extraction, & Analysis

#### Transcription Factor Binding Sites: Prediction & Evaluation

#### Analyzing Phenotype & Fitness Data

#### Mapping Omics Data to Pathways

---
### Module 4 - Mining and Predicting 3D Protein Structures

(insert diagram of workflow)
**Objective Summanry**
- Visualize 3D protein structures
- Build models of protein structures
- Identify ligand-binding pockets
- Predict Protein-Protein or Protein-Nucleic acid binding interfaces

#### Visualizing 3D Protein Structures

#### Building Models of Protein Structures

#### Identifying Ligand-binding Sites

#### Predicting Protein-Protein, Protein-NA Binding Interfaces

---
### Module 5 - Data Visualization ***(more specific?)***

(insert diagram of workflow)
**Objective Summary**
- Discuss different network visualization tools
- Visualize gene neighborhoods
- Compare sequence Logos
- Add different types of data to phylogenetic trees

#### Network Visualization Tools

#### Gene Neighborhood Analysis

#### Comparing Sequence Logos

#### Mapping Data to Phylogenetic Trees (Phylogenomics)
