---
title: trimAl
description: trimAl: a tool for automated alignment trimming
published: true
date: 2020-08-19T15:35:13.157Z
tags: tool, sequence alignment, evolution, comparative genomics, data visualization, protein, conservation, toolbox, alignment
editor: markdown
---

# trimAl

> trimAl is a tool for automated alignment trimming. Its speed and the possibility for automatically adjusting the parameters to optimize the phylogenetic signal-to-noise ratios for different families, makes trimAl especially suited for large-scale phylogenomic analyses, involving thousands of large multiple sequence alignments.
>
> trimAl has been implemented in C++ programming language. So far a command-line version to download and a web interface are available.
{.is-info}

 

## Website 

- [trimAl *Main Page*](http://trimal.cgenomics.org/introduction)
- [trimA1 Webserver *trimAI v1.2 via Phylemon2*](http://phylemon2.bioinfo.cipf.es/)
 {.links-list}


## Notes from the user
 
- Users can create a Phylemon account and store data for future use on the webserver. (The webserver can also be used without an account.)

## Ease of use

- The tool is not very intuitive, but users can find instructions or examples for most purposes. 

## Help

- [trimAI Help *Webserver Interface Instructions*](http://trimal.cgenomics.org/use_of_the_trimal_v1.2_webserver_interface)
- [trimAI FAQ *FAQ Page*](http://trimal.cgenomics.org/faq)
 {.links-list}


## Citation 

- Capella-Gutiérrez, S., Silla-Martínez, J. M., & Gabaldón, T. (2009). [trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses.](https://academic.oup.com/bioinformatics/article/25/15/1972/213148) Bioinformatics, 25(15), 1972-1973.
{.grid-list}



