---
title: PDBeMotif
description: PDBeMotif is an extremely fast and powerful search tool that facilitates exploration of the Protein Data Bank (PDB) by combining protein sequence, chemical structure and 3D data in a single search.
published: true
date: 2020-07-16T15:28:42.306Z
tags: tool, proteins, comparative genomics, data capture, database, browser, data visualization, mapping, protein family, data export, conservation, model, protein domain, binding, interaction, motif, structural analysis, analysis tool
editor: markdown
---

# PDBeMotif

>    PDBeMotif is an extremely fast and powerful search tool that facilitates exploration of the Protein Data Bank (PDB) by combining protein sequence, chemical structure and 3D data in a single search. Currently it is the only tool that offers this kind of integration at this speed. PDBeMotif can be used to examine the characteristics of the binding sites of single proteins or classes of proteins such as Kinases and the conserved structural features of their immediate environments, either within the same specie or across different species. 
&NewLine;
The prediction of the effect of modifications to small molecules that bind to the active and/or regulatory sites of proteins on their efficacy can be based on the outcome of analytic work done using PDBeMotif.
{.is-info}



## Website

- [PDBeMotif *Main Page*](https://www.ebi.ac.uk/pdbe-site/pdbemotif/)
{.links-list}


## User Notes

- Users need to use Java applet-enabled web browsers to view 3D motif and ligand binding structural information.
- Users can search for specific motifs, for motifs within specific proteins, by small molecules or ligands, or perform a multi-criteria search by combining different searches into one.

## Ease of Use

- This tool is not immediately intuitive, but has a fairly thorough help guide.  Users will likely need to play around with the tool for some time before grasping its full capacity. 

## Help

- [PDBeMotif Help *Examples and Guide*](https://www.ebi.ac.uk/pdbe-site/pdbemotif/help/help.jsp?topic=eg)
{.links-list}

## Citations

- Golovin A, Henrick K: [MSDmotif: exploring protein sites and motifs.](https://link.springer.com/article/10.1186/1471-2105-9-312) BMC Bioinformatics 2008, 9:312.
- A. Golovin, K. Henrick: [Chemical Substructure Search in SQL,](https://pubs.acs.org/doi/abs/10.1021/ci8003013) J.Chem. Inf. Model., 2009, 49 (1), 22-27.
-	Golovin A., Dimitropoulos D., Oldfield T., Rachedi A. and Henrick K. (2005) [MSDsite: A Database Search and Retrieval System for the Analysis and Viewing of Bound Ligands and Active Sites.](https://onlinelibrary.wiley.com/doi/abs/10.1002/prot.20288) PROTEINS: Structure, Function, and Bioinformatics 58(1): 190-9.
{.grid-list}