---
title: String
description: Protein-Protein Interaction Networks and Functional Enrichment Analysis
published: true
date: 2020-08-08T20:26:44.942Z
tags: annotation, archaea, bacteria, tool, cog, functional association, data capture, homolog discovery, analysis tools, database, literature, homology, co-expression, data visualization, prediction, protein, clustering, protein family, co-occurrence, data export, eukaryota, curated, network, protein domain, binding, interaction, protein-protein
editor: markdown
---

# STRING

> STRING is a database of known and predicted protein-protein interactions. The interactions include direct (physical) and indirect (functional) associations; they stem from computational prediction, from knowledge transfer between organisms, and from interactions aggregated from other (primary) databases. STRING data is derived from 5 main sources: Genomic context predictions, High-throughput lab experiments, Conserved co-expression, Automated textmining, previously published knowledge in databases.
{.is-info}

## Website

- [STRING *Main Page*](http://string.embl.de/)
- [STRING Help *Help Page*](https://string-db.org/cgi/help.pl?sessionId=UUAtCeLj4DQr)
{.links-list}

## User Notes

- You can search for both individual proteins and protein families.
- Proteins with Values/Ranks - Functional Enrichment Analysis: with this search feature you can submit your experimental data to perform functional enrichment analysis.
- In the "Settings" tab you can choose whether your network edges (which represent the predicted functional associations) are shown as type of evidence, confidence strengths, or predicted molecular actions. This is also where you adjust your confidence limit and choose which types of evidence you want to use in your search (for example, you can exclude textmining evidence).
- The default view of your search is of the interaction network but much more information can be gathered by switching your viewer, clicking on nodes and edges, exploring the "Legend" and "Analysis" tabs, etc (see "STICH Help" page).
- Beware of paralogs, all functional associations of a family are lumped together.

## Ease of Use

This website can be used very quickly and easily to gather a wide range of information.
{.is-info}

## Citations

When using STRING, please consult (and cite) the following references:

- Szklarczyk D, Gable AL, Lyon D, Junge A, Wyder S, Huerta-Cepas J, Simonovic M, Doncheva NT, Morris JH, Bork P, Jensen LJ, von Mering C. [STRING v11: protein-protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets.](https://www.ncbi.nlm.nih.gov/pubmed/30476243) Nucleic Acids Res. 2019 Jan; 47:D607-613.
&NewLine;
-	Szklarczyk D, Morris JH, Cook H, Kuhn M, Wyder S, Simonovic M, Santos A, Doncheva NT, Roth A, Bork P, Jensen LJ, von Mering C. [The STRING database in 2017: quality-controlled protein-protein association networks, made broadly accessible.](https://www.ncbi.nlm.nih.gov/pubmed/27924014) Nucleic Acids Res. 2017 Jan; 45:D362-68.
&NewLine;
-	Szklarczyk D, Franceschini A, Wyder S, Forslund K, Heller D, Huerta-Cepas J, Simonovic M, Roth A, Santos A, Tsafou KP, Kuhn M, Bork P, Jensen LJ, von Mering C. [STRING v10: protein-protein interaction networks, integrated over the tree of life.](https://www.ncbi.nlm.nih.gov/pubmed/25352553) Nucleic Acids Res. 2015 Jan; 43:D447-52.
&NewLine;
-	Franceschini A, Lin J, von Mering C, Jensen LJ. [SVD-phy: improved prediction of protein functional associations through singular value decomposition of phylogenetic profiles.](https://www.ncbi.nlm.nih.gov/pubmed/26614125) Bioinformatics. 2015 Nov; btv696.
&NewLine;
-	Franceschini A, Szklarczyk D, Frankild S, Kuhn M, Simonovic M, Roth A, Lin J, Minguez P, Bork P, von Mering C, Jensen LJ. [STRING v9.1: protein-protein interaction networks, with increased coverage and integration.](https://www.ncbi.nlm.nih.gov/pubmed/23203871) Nucleic Acids Res. 2013 Jan; 41:D808-15.
&NewLine;
-	Szklarczyk D, Franceschini A, Kuhn M, Simonovic M, Roth A, Minguez P, Doerks T, Stark M, Muller J, Bork P, Jensen LJ, von Mering C. [The STRING database in 2011: functional interaction networks of proteins, globally integrated and scored.](https://www.ncbi.nlm.nih.gov/pubmed/21045058) Nucleic Acids Res. 2011 Jan; 39:D561-8.
&NewLine; 
-	Jensen LJ, Kuhn M, Stark M, Chaffron S, Creevey C, Muller J, Doerks T, Julien P, Roth A, Simonovic M, Bork P, von Mering C. [STRING 8--a global view on proteins and their functional interactions in 630 organisms.](https://www.ncbi.nlm.nih.gov/pubmed/18940858) Nucleic Acids Res. 2009 Jan; 37:D412-6.
&NewLine;
-	von Mering C, Jensen LJ, Kuhn M, Chaffron S, Doerks T, Krueger B, Snel B, Bork P. [STRING 7--recent developments in the integration and prediction of protein interactions.](https://www.ncbi.nlm.nih.gov/pubmed/17098935) Nucleic Acids Res. 2007 Jan; 35:D358-62.PubMed
&NewLine; 
-	von Mering C, Jensen LJ, Snel B, Hooper SD, Krupp M, Foglierini M, Jouffre N, Huynen MA, Bork P. [STRING: known and predicted protein-protein associations, integrated and transferred across organisms.](https://www.ncbi.nlm.nih.gov/pubmed/15608232) Nucleic Acids Res. 2005 Jan; 33:D433-7.
&NewLine; 
-	von Mering C, Huynen M, Jaeggi D, Schmidt S, Bork P, Snel B. [STRING: a database of predicted functional associations between proteins.](https://www.ncbi.nlm.nih.gov/pubmed/12519996) Nucleic Acids Res. 2003 Jan; 31:258-61.
&NewLine;  
-	Snel B, Lehmann G, Bork P, Huynen MA. [STRING: a web-server to retrieve and display the repeatedly occurring neighbourhood of a gene.](https://www.ncbi.nlm.nih.gov/pubmed/10982861) Nucleic Acids Res. 2000 Sep 15;28(18):3442-4.
{.grid-list}