---
title: Class Microbial Comparative Genomics MCB6318 
description: Overview of the tools and websites covered in the "Microbial Comparative Genomics" class (MCB6318)
published: true
date: 2024-02-03T14:58:37.366Z
tags: recommended, favorites, workflow, teaching
editor: markdown
dateCreated: 2023-03-01T20:05:36.379Z
---

# MCB6318 - Comparative Microbial Genomics
> This class gives basic tools to capture knowledge on microbial protein families and reviews intergrative datamining methods that allow to predict the function of proteins.  The focus is mainly  on bacterial proterins and metabolism. But some eukaryotic resources are also given as some protein families are widely conserved. All tools used in this class are web-based. The mumber of web-based biological database is very large >6000 and growing. The [Data Commons](https://ngdc.cncb.ac.cn/databasecommons/) database is a great resource to search for ones that fit your needs, knowing that many databases are [short lived](https://journal.embnet.org/index.php/embnetjournal/article/view/803/1209). In this course, are list the ones we use the most in our research and teaching.
{.is-info}

### Module 1 - Dealing with the Avalanche of Genomic Data
> With the genomic revolution started in 1995, microbiology has become a data science as all aspects of the field are driven by knowledge derived from sequences ( see extra reading [here](https://vdclab-wiki.herokuapp.com/en/courses/MCB6318/M1-additional-reading)). With >400,000 genomes sequenced ( see current number on the [GOLD database](https://vdclab-wiki.herokuapp.com/en/databases/data-integration/JGI-GOLD)), gathering informations on genomes and proteins is not easy.
{.is-info}

**Module Objectives**
- Extract and analyze bacterial genomes of interest 
- Extract sets of proteins/genes sequences of a specific family
- Find the literature associated with a protein family
- Map accession numbers from one database to another
- Reformat Fasta files using REGEX  tools

#### Extracting and Analyzing Bacterial Genomes

> Many integrated databases are available to identify or gather genomic bacterial or archaeal sequences. We use BV-BRC routinely but IMG/JGI, Microscope and NCBI are also great resources.
{.is-info}

- [**BV_BRC**](/databases/bacterial_databases/patric)
- [**IMG/JGI**](/databases/prokaryote_databases/JGI-IMG)
- [**NCBI Datasets**](https://www.ncbi.nlm.nih.gov/datasets/)
- [**Microscope**](/tools/analysis/Genoscope-Microscope)
### Notes 
Some genomes might be not correctly classified. Focusing on reference and/or complete genomes is always better when you are gathering genomes. 

#### Extracting and Analyzing eukaryotic & plants Genomes
> For comparative purposes knowing useful fungal or plant databases is sometimes very useful  .
{.is-info}
- [**SGD** *Baker's yeast*](/databases/organism_specific/SGD)
- [**PomBase** *Fission yeast*](/databases/organism_specific/PomBase)
- [**Plaza** *Plants*](/platform/plant-comparative-genomic-tools/PLAZA)
- [**EupathDB** *Eukaryotic MIcrobes*](/databases/data-integration/EuPathDB)


#### Extracting Sets of Protein/Gene Sequences for a Target Family
> Using [BlastP](https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastp&PAGE_TYPE=BlastSearch&LINK_LOC=blasthome) @[NCBI](https://www.ncbi.nlm.nih.gov/) is not the most efficient way to gather diverse set of sequences because you will have to many very similar sequences unless you limit greatly your genomes or your database. [HHmer](https://www.ebi.ac.uk/Tools/hmmer) can help do sensitive searches more compactly. Our favorite methods to gather sequences from a protein family quickly are [orthology databases](/resources/general_resources/Quest-for-Orthologs) or [Protein Family databases](https://vdclab-wiki.herokuapp.com/t/protein%20family). One must still be conscious of paralogs and fusions. Uniprot is a great starting point as it federates many other resources. The ones we use frequently are listed below.
{.is-info}

**Orthology databases**
- [**Uniprot**](/UniProtKB-Advanced-Search)
- [**OrthoInspector**](/phylogeny/phylogenetic-distribution/OrthoInspector)
- [**OMA**](https://omabrowser.org/oma/home/)
- [**KEGG KO**](/databases/ortholog/KEGG-OC)

**Protein family databases**
- [**EggNog**](http://eggnog6.embl.de)
- [**InterPro**](/databases/general_databases/InterPro)
- [**CATH**](/databases/data-integration/CATH)
- [**AnnoTree**](/phylogeny/phylogenetic-distribution/AnnoTree)

#### Finding Literature Pertaining to a Target Protein Family

> Finding the papers published on different members of the same protein family can be quite difficult as explained in detail in this manuscript [Reed et al](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10187207/). The main message is to always start with Paper Blast with a Family input (HMM) and also use different phylogenetically diverse sequences as input.  
{.is-info}

- [**PaperBlast** *Single Sequence input*](/literature-search/PaperBLAST)
- [**PaperBlast** *Family Input*](https://papers.genomics.lbl.gov/cgi-bin/hmmSearch.cgi)
- see also [**VdcLab Wiki Litterature links**](https://vdclab-wiki.herokuapp.com/en/literature-search) 

#### Mapping Accessions/Identifiers Between Different  Databases
> Mapping Ids from one database format to the other is a constant issue when navigating database. Mappers are available but their use is  can be tricky and fustrating.
{.is-info}

- [**DAVID**](https://david.ncifcrf.gov/conversion.jsp)
- [**UniProt Mapper**](https://www.uniprot.org/id-mapping)
- [**BV-BRC Mapper**](https://www.bv-brc.org/app/IDMapper)
- [**KEGG Database mapping**](https://www.genome.jp/linkdb/)
- [**BridgeDB**](https://www.bridgedb.org/)
- [**Metabolite conversion at MetaboAnalyst**](https://www.metaboanalyst.ca/MetaboAnalyst/upload/ConvertView.xhtml)

#### Reformatting FASTA Files Using REGEX Tools
> To do multiple alignments and phylogenetic trees, the headers in Fasta sequences will often need to be reformated. A user with little programming experience can use regular expression manipulations (RegEx) to do that pretty easily.
{.is-info}

[**RegExr**](https://regexr.com)


------------------------------------------------------------------------

### Module 2 - Linking Gene and Function
> In the genomic era, protein function is predicted using automated methods during the genome annoation process ( see review by [Stein](https://pubmed.ncbi.nlm.nih.gov/11433356/) that is still very current). Many different types of errors can occur . See List [here](https://vdclab-wiki.herokuapp.com/en/courses/MCB6318/annotation-errors). One of the ways to check for errors is to put the annotation in its biological context by performing metabolic reconstructions. It is also a path to identify missing genes or pathway holes. Comparative genomic methods such as physical clustering  or gene neighborhood analyses methods can then identify candidates for these missing genes ( see Review [here](https://pubmed.ncbi.nlm.nih.gov/20001958/)) and - [Additional Reading here](https://vdclab-wiki.herokuapp.com/en/courses/MCB6318/M2-additional-reading). Automation of functional anntotation requires controled vocabularies that describes functions such as Gene Ontologies (GO) and/or Enzyme Commission (EC) numbers as described [here](https://vdclab-wiki.herokuapp.com/en/courses/MCB6318/controlled-vocabularies). 
{.is-info}

**Module objectives**
- Place an annotated gene in a biological context
- Check if a metabolic pathway is complete in a sequenced organism
- Visualize and compare pathway information in different databases
- Identify pathway holes
- Conduct phylogenetic distribution queries
- Analyze physical clustering

#### Exploring Pathway databases
> Databases that compile patwhay information and place predicted proteins in specific pathways starting with genomic information can be used to perform metabolic reconstruction, check if a pathway is complete and compare pathways between organisms. KEGG has both Pathways and Modules even if the latter have been also integrated in the former. Metacyc/BioCyc use now a subscription model. 
{.is-info}
- [**KEGG Pathways**](https://vdclab-wiki.herokuapp.com/en/databases/enzyme-pathway/KEGG-PATHWAY)
- [**KEGG Modules**](https://www.genome.jp/kegg/module.html)
- [**MetaCyc/BioCyc**](https://vdclab-wiki.herokuapp.com/e/en/databases/enzyme-pathway/MetaCyc)

> General genome integration databases also have tools to perform metabolic reconstructions and they have nice features to compare pathways between genomes. 
{.is-info}
- [**IMG**](https://img.jgi.doe.gov/cgi-bin/m/main.cgi)
- [**BV-BRC**](https://www.bv-brc.org/app/ComparativeSystems)
- [**Microscope**]((https://mage.genoscope.cns.fr/microscope/metabolism/index.php?))

#### Finding candidates for "missing genes" or pathway holes
> Once you have identified a globally or missing gene case  by doing metabolic reconstruction, or through literature searches or looking at the [Unknown Gene/Enzyme Databases](https://vdclab-wiki.herokuapp.com/en/databases), you can use different types of genomic or post-genomic derived associations to  find gene candidates. In this module we focus on protein fusions,  physical clustering and phylogenetic profiles, in the next modules we will look at co-regulation, protein interactions  and omics derived associations. All metabolic pathways have pathway holes and non-orthologous displacments that fill these holes. [See this analysis of the Folate pathway](https://pubmed.ncbi.nlm.nih.gov/25210598/).
{.is-info}

>Integrative database. The most user friendly database to integrate differnt types of association evidence is STRING. It is a good strating point but as everything is pre-computed. PubSEED has a great physical clustering tool but the genomes have not been update in many years. This tools is now integrated in [BV-BRC](/databases/bacterial_databases/patric).
{.is-info}
- [**String**](https://vdclab-wiki.herokuapp.com/en/tools/analysis/String)
- [**PubSEED**](https://vdclab-wiki.herokuapp.com/en/databases/bacterial_databases/pubseed)
- [**MBGD**](/databases/ortholog/MBGD)

> Additional gene clustering platform. There are many tools to explore physical clusteringa associations. The ones we use the most in addition to intergrative databases are listed below.
{.is-info}

- [**Fastgenomics** *very quick fisrt overview but only 6000 genomes*](https://fast.genomics.lbl.gov/cgi/search.cgi)
- [**GeCoViz** *nice graphics based o EggNog*](https://gecoviz.cgmlab.org)
- [**EFI GNN** *Most extensive in terms of data, all Uniprot*](https://efi.igb.illinois.edu/efi-gnt/)
- [**ProFaNA**](http://bioinfo.sggw.edu.pl/neighborhood_analysis/)

> Fusion databases. It is difficult to differentiate between true fusion (or Rosetta stone proteins) from multifomain proteins. The former have to be found as seperate domains in some organisms. Model SEED is a specific  fusion database but otherwise general domain database have indirectly fusion information.
{.is-info}

- [**CDD @NCBI**](https://www.ncbi.nlm.nih.gov/Structure/lexington/lexington.cgi)
- [**InterPRO**](https://vdclab-wiki.herokuapp.com/en/databases/general_databases/InterPro)
- [**EggNog**](http://eggnog6.embl.de)
- [**Model SEED Fusion**](https://modelseed.org/projects/fusions/)

> Phylogenetic Distribution. The tools to  extract the  protein family are present and absent in specific organisms are integrated in the general integrative genome databases. They vary by the number and taxonomy of genomes. The difficulty lies in choosing enough genomes to  have a reasonable of candidates. 
{.is-info}

- [**Microscope Phyloprofiler**](https://mage.genoscope.cns.fr/microscope/compgenomics/phyloprofil.php)
- [**MicrobesOnline Phyloprofiler**](http://www.microbesonline.org/cgi-bin/matchphyloprofile.cgi)
- [**JGI/IMG Phyloprofiler**](https://img.jgi.doe.gov/cgi-bin/m/main.cgi?section=PhylogenProfiler&page=phyloProfileForm)
- [**OrthoMCL**](https://orthomcl.org/orthomcl/app/#phyleticPattern)

#### AI tools to predict protein functions
> There is currently an explosion on the use of AI in biology applications.  The field of structural biology has been transformed by alpha-fold. In terms of predicting function, AI tools are helping to capture and propapagate functional terms  but can still make overpredictions and are not designed to discover novel function yet as discussed [HERE](https://www.biorxiv.org/content/10.1101/2023.12.18.571875v1.abstract).   
{.is-info}

- [**Predicting EC number from sequence**](https://clean.platform.moleculemaker.org/configuration)
- [**Predicting Gene Ontology from sequence**](https://google-research.github.io/proteinfer/)
- [**Predicting differnt types of protein information** ](https://predictprotein.org/)

----------------------------------------------------------------------------------
### Module 3 - Genome-wide Analysis of Experimental Data and Data Integration
> In the last 20 years different types of "omics" data  (transcriptomics, proteomics, metabolomics to name the most common) can be generated and the amount is increasing at an exponential rate as the costs go down and the technology improves. In parallel, predictions of regulatory network based on motif analyses has improved and when combined with transcriptomic data can help identify genes that are co-regulated. One problem with all these data sets is that they are noisy and require replicates ( from 3 to 6 depending on the techniques).  Also, Id mapping issues make the use of web-based analysis tools sometimes fustrating. Many of the best tools to analyze multi-omics data are R packages but there are few web-based alternatives to both analyze your own and published data. In front of this data deluge, a useful to keep track is [OmicsDI](https://www.omicsdi.org/). See addional reading [Here](https://vdclab-wiki.herokuapp.com/en/courses/MCB6318/M3-additional-reading). 
{.is-info}

**Module Objectives**
- Describe the overall process used to generate and analyze gene expression data
- Extract and analyze gene expression data from databases
- Predict and evaluate the presence of DNA binding sites for specific transcription factors
- Analyze Phenotype/Fitness data
- Analyze Protein/Protein interaction data
- Map omics data to pathways

#### Gene Expression Data: Generation, Extraction, & Analysis
>  Transcriptomics data was initially captured using Microarrays and is now mainly generated using RNAseq. The tools to analyze the RNASeq data generated by experimentalists have matured and are available in pipelines that are quite easy to use. Understanding the steps and statistics in these analysis flow is still very important both to design and interpret experiments as discussed in the papers HERE (to add). Analysing the published RNASeq data is still a challenge even if it is all publicly available in GEO. Specialized databases have emerged to help navigate the data deluge, some organisms specific, some for group of organisms, but the bulk of the resources are for eukaryotes (mainly human). For Bacteria webtools to identify  co-expressed genes or tools to map transcriptomic data to genome browsers are available for specific genomes or group of genomes but these are very community specific.  A subset of examples that we use routinely are given below. 
{.is-info}

**Platforms to analyze your microbial RNASeq data ( Free and web-based)**
- [**RNASeq worflow**](https://docs.kbase.us/workflows/rnaseq)
- [**BV-Brc RNASeq**](https://www.bv-brc.org/app/Rnaseq)
	
**Multi Species Depositories and/or analyses**
- [**GEO2R** *All transcriptomic data must be deposited in GEO before publication*](/databases/microarray-rnaseq/NCBI-GEO)
- [**OmicsDI** *Compilation of all types of omics data*](https://www.omicsdi.org/)
- [**Atted-II** *Plant expression data*](/databases/microarray-rnaseq/ATTED-II)
- [**CoExpressDB** * Model Eukaryotes expression data*](https://coxpresdb.jp/)
- [**iModulon** *Different Bacteria including E. coli and B. subtilis*](/omics-data/omics-data/iModulon)
- [**GeneFriends**](http://www.genefriends.org)
- [**Microscope**](https://mage.genoscope.cns.fr/microscope/expdata/NGSProjectRNASeq.php?&wwwpkgdb=459f81b9d7971022da9124ae1c1df36e)

**Organism Specific databases**
- [**GeneXP** *E. coli expression data*](https://genexpdb.okstate.edu/)
- [**PneumoExpress** *Streptococcus pneumoniae expression data*](https://vdclab-wiki.herokuapp.com/en/omics-data/omics-data/PneumoExpress)
- [**SPELL** *S. cerevisiae expression  data*](/databases/organism_specific/SPELL)
- [**Subtiwiki** *Contains expression data for B . subtilis*](/databases/organism_specific/SubtiWiki)	
- [**RegulonDB** *Contains expression data for E. coli*](/databases/transciription_factor/RegulonDB)
- [**TBdatabase** *Mycobacterium tuberculosis systems biology*](http://tbdb.bu.edu/tbdb_sysbio/RegulatoryNetwork.html)
- [**SalCom** *Salmonella expression data*](/databases/organism_specific/SalCom)	
- [**CyanoExpress** *Cyanobacteria expression data*](/databases/organism_specific/CyanoEXpress)
- [**SEEK** *Human expression data*](/databases/organism_specific/SEEK)	

**HTS data mapped to browser**

- [Different Pathogens HTS data mapped to genome](https://wadsworth.org/research/scientific-resources/interactive-genomics)
- [E. coli HTS data mapped to genome](https://www.nichd.nih.gov/research/atNICHD/Investigators/storz/data-protocols)

#### Bacterial transcription Factor Binding Sites: Prediction & Evaluation
> Co-regulation can be predicted by the identification of the operator sites for specific regulators. These can be predicted using in silico methods or by combining with expression data.  
{.is-info}

- [**Regprecise** *Computed bacterial regulons*](/databases/transciription_factor/RegPrecise)
- [**Prodoric** *Computed bacterial regulons*](/databases/bacterial_databases/PRODORIC)
- [**REgulonDB** *E. coli regulation centric*](/databases/transciription_factor/RegulonDB)
- [**iModulon** *Regulatory networks for different Bacteria*](/omics-data/omics-data/iModulon)
- [**Prochipdb** *Chipseq data analysis*](https://prochipdb.org/)

#### Analyzing Phenotype & Fitness Data
> Ordered mutant libaries have been constructed for a dozen model organisms. This allowed to identify essential genes  and screen these mutants in a wide variety of growth conditions including the presence of many chemicals. More recently TnSeq TnSeq-derived fitness data or [CRISPRi knockout data](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8510551/) can be generated more quickly. Accessing the phenome data can be sometimes difficult (for example the data from the *E.coli* Carol Gross [data set](https://pubmed.ncbi.nlm.nih.gov/21185072/) is no longer available.
{.is-info}

- [**OGEE** *essentiality data*](/databases/phenotypes-fitness/OGEE)
- [**DEG** *essentiality data*](/databases/data-integration/DEG)
- [**FitnessBrowser** *TnSeq Data*](/databases/phenotypes-fitness/FitnessBrowser)
- [**FitDB** *S. cerevisiae Phenome data*](/databases/phenotypes-fitness/FitDB)
- [**Yeast Phenome** *S. cerevisiae Phenome data*](https://www.yeastphenome.org/)

#### Protein/Protein Interaction data
> Protein/protein interaction data was one the first type of HTS data to be collected using different type of approaches as discussed [HERE](https://pubmed.ncbi.nlm.nih.gov/37450113/). It is very noisy but can be useful when combined with other types of association evidence. This data has been linked in every Uniprot entry and  integrated in multiomics databases such as STRING see below, but more specific databases are discussed [HERE](https://pubmed.ncbi.nlm.nih.gov/26077899/) and a few listed below.  
{.is-info}
- [**DIP**](https://dip.doe-mbi.ucla.edu/dip/Main.cgi)
- [**BioGrid** *Integrates protein, genetic and chemical intersctions*](https://thebiogrid.org/)
- [**Intact**](https://www.ebi.ac.uk/intact/home)
- [**CPDB** *only human, mouse and yeast*](http://cpdb.molgen.mpg.de/)


#### Data Integration and Mapping Omics Data to Pathways
> Not many webtools allow to map omics data to pathways specifically for Bacteria (see [Review](https://academic.oup.com/bib/article/23/6/bbac435/6761962)). A few databases generate very user friendly association netwoks that compile all types of data includeing Omics data.
{.is-info}

**Advanced data queries**
- [**YeastMine**](https://yeastmine.yeastgenome.org/yeastmine/begin.do)
- [**InterMine**](/tools/gene-finders/InterMine)

** Precomputed Networks that integrate different types of data** 
- [**String**](/tools/analysis/String)
- [**Stich**](http://stitch.embl.de/)
- [**GeneFriends**](https://www.genefriends.org/)
- [**EcoliNet**](/tools/gene-networks/EcoliNet)	 
- [**GeneMania**](/tools/gene-networks/GeneMANIA)
- [**FunCoup**](https://funcoup.org/search/)

**Mapping different types of data to pathway maps** 
- [**Impres**  *yeast maize and human*](https://impres.missouri.edu/impres)
- [**Paintomics** *Visualize differnt types of data on pathways*](/tools/pathways/PaintOmics)
- [**Reactome** *Mainly model eukaryotes*](/databases/data-integration/Reactome) 
- [**Impala**](http://impala.molgen.mpg.de/)
- [**iPath3**](https://pathways.embl.de/)
- [**ExpressAnalyst**](https://www.expressanalyst.ca/ExpressAnalyst/home.xhtml)
- [**3omics** *Human only* ](https://3omics.cmdm.tw/)
- [**MetaboAnalyst** *model organisms*](https://www.metaboanalyst.ca/MetaboAnalyst/home.xhtml)
- [**BioCyc** ](/databases/data-integration/BioCyc) 
Note: BioCyc is subscrition based (The instructor gets educational access for the duration of the class)

------------------------------------------------------------------------------
### Module 4 - Mining and Predicting 3D Protein Structures
> .
{.is-info}

**Module Objectives**
- Visualize 3D protein structures
- Build models of protein structures
- Identify ligand-binding pockets
- Predict Protein-Protein or Protein-Nucleic acid binding interfaces

#### Visualizing 3D Protein Structures
> Many databases have integrated structure viewers such as [PDB](https://vdclab-wiki.herokuapp.com/en/structure/visualization-platforms/RCSB-PDB) or [iCn3D](/structure/visualization-platforms/iCn3D). These are sufficient to gather initial information about ligand bindong site or fold membership or structural similarity but to do more advanced structural analysis two platforms are available Pymol or ChimeraX. [Pymol](https://vdclab-wiki.herokuapp.com/en/visualization/structure_platform/PyMOL) requires a subscription even if educational access is available but ChimeraX is free and quite easy to learn.
{.is-info}

- [**ChimeraX**](https://www.cgl.ucsf.edu/chimerax/)

#### Building Models of Protein Structures
> Protein  prediction tools  had progressed greatly in the last 10 years with the [**CASP**](https://predictioncenter.org/) competition pushing the modelers to build to  tools like [**Phyre**](https://vdclab-wiki.herokuapp.com/en/structure/visualization-platforms/Phyre). But the release  in 2020 of  [**AlphaFold**](https://alphafold.com/)  has been a game changer for the field. Alphafold predictions have been now been integrated for every Uniprot entry and new development are beeing published constantly as discussed [**HERE**](https://www.nature.com/articles/d41586-022-00997-5)
{.is-info}

#### Identifying Ligand-binding Sites
> The nature and  position of the ligand(s) is critical to  understand protein function.
{.is-info}

- [**BindWeb**](http://www.csbio.sjtu.edu.cn/bioinf/BindWeb/index.html)
- [**GalaxySite**](http://galaxy.seoklab.org/cgi-bin/submit.cgi?type=SITE)
- [**FindSite-Comb2**](http://pwp.gatech.edu/cssb/FINDSITE-COMB-2)
- [**Coach**](https://zhanglab.ccmb.med.umich.edu/COACH/)
- [**eF-Seek**](https://pdbj.org/eF-seek/top.do)

#### Predicting Protein-Protein, Protein-nucleic acids Binding Interfaces
> Alpha-fold is not yet optimal to  model interactions between macromolecules even if it is a very active field of study as discussed [HERE](https://www.nature.com/articles/s41467-022-28865-w). The Kurgan lab has a suite of interesting tools  in the  [BioMine servers](http://biomine.cs.vcu.edu/) including many nucleic acid binding tools. Unfortunately these tools are not very reliable yet and the servers that provide them come and go. This [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9585174) reviews Protein/RNA binding prediction tools.
{.is-info}

**Protein/Protein Interaction Predictions**
- [**INGA**](https://inga.bio.unipd.it/)
- [**PPISP**](https://pipe.rcc.fsu.edu/meta-ppisp.html)

**Protein/RNA or Protein/DNA Interaction Predictions**
- [**NCBRPred** *predicting RNA binding*](http://bliulab.net/NCBRPred/server)
- [**hybridDBRpred** *predicting DNA binding*](http://biomine.cs.vcu.edu/servers/hybridDBRpred/)
- [**DRNApred** *predticting RNA and DNA binding*](http://biomine.cs.vcu.edu/servers/DRNApred/)
- [**RBP2GO** *database that compiles experimental Protein/RNA binding*](https://RBP2GO.DKFZ.de)
- [**DESCRIBESPROT** *runs a combination of predition tools*](http://biomine.cs.vcu.edu/servers/DESCRIBEPROT/main.php)

### Module 5 - Data Visualization examples 
> Tools to visualize gene and protein data are critical to both exploring and publications. A seminal review on datavisualization can be found [HERE](https://www.annualreviews.org/doi/10.1146/annurev-biodatasci-080917-013424). More generally the [VizBI](https://vizbi.org/About) community is very active in developping tools. For the purpose of this class we will just focus on a few types of tools.
{.is-info}

**Module Objectives**

- Visualize gene neighborhoods
- Compare sequence logos
- Add different types of data to phylogenetic trees

#### Gene Neighborhood (GN) Analysis
 > Depending on the explicit goals, explore neigbhorhhods to find associations or create figures for a manuscript, the choice of the tool might be different. Many integrated database such as String, IMG-JGI or BV-BRC have GN tools.  They can be useful for exploration but not for figures.  The most extensive for exploration is the [EFI/GNT](https://efi.igb.illinois.edu/efi-gnt/).In the last years a flurry of user frienfly tools have been published and should be explored. The ones we use often are listed below.
{.is-info}

-[**GeneGraphics**](https://v2.genegraphics.net/)
-[**Gizmogene**](http://www.gizmogene.com/)
-[**GeneCoViz**](https://gecoviz.cgmlab.org/)
-[**FastGenomics**](https://fast.genomics.lbl.gov/cgi/search.cgi)
-[**CoreWiki**](http://corewiki.uni-goettingen.de/welcome)

#### Comparing Sequence Logos
> TEXT to do.
{.is-info}

-[Compare2LOGOs](http://www.twosamplelogo.org/)

#### Mapping Data to Phylogenetic Trees (Phylogenomics)
> TEXT to do.
{.is-info}

iTOL
PhyloT
